Content: Remnant Expanded Horizons Astral (#5367)
endless-sky/endless-sky: data/remnant/remnant missions.txt

event "remnant: cognizance timer"
	system "Nenia"
		fleet "Ember Waste Node" 6000

mission "Remnant: Expanded Horizons Astral 1"
	name "Preparation for expedition"
	description "To prepare for an expedition to Sagittarius A*, go to <destination> with 6 outfit space available."
	passengers 1
	minor
	to offer
		has "Remnant: Expanded Horizons Quarg 3: done"
		# random < 25
	source
		government "Remnant"
	destination
		government "Remnant"
	on offer
		conversation
			`"In the starport cafeteria you come across Dawn sitting at a table looking worried. She smiles when she sees you, and waves you to the seat opposite her. "Back from gallivanting around the galaxy with more tales to share?" she signs.`
			choice
				`	"Yes, as a matter of fact..."`
				`	"No, just hunting Korath..."`
				`	"No, just testing some stuff..."`
				`	"You look worried. Is something happening?"`
					goto worried
			`	She listens for a minute or two as you tell her about what you have been up to, but it is obvious that she has something on her mind.`
			choice
				`	"You look worried. Is everything alright?"`
					goto worried
				`	"You look distracted. We can talk later."`
					defer
			label worried

> 			`	She makes a gesture of appreciation. "Yes, I'm worried, <first>. I keep feeling like there's something I'm missing." She pauses, then continues. "I should back up a step. While I do not have a permanent position yet, I am currently serving with an astrophysics team. We have been studying a variety of things, and I have been tasked with reviewing our bulk astrophysical data. It is fairly routine,  but I keep getting this prickly feeling that there's a problem that I am missing. Nothing obvious, of course. Our neural networks would pick up something obvious or repetitious really quickly. But the feeling just will not go away."`

Reviewer: > "Yes, I'm worried, <first>. I keep feeling like there's something I'm missing." She pauses, then continues. "I should back up a step. While I do not have a permanent position yet, I am currently serving with an astrophysics team.
"She pauses, then continues," feels like it's written for the sake of being written. More description would be good.

Author: Hmmm... I'll change up that a bit. I keep going over it, not sure what to do. But we'll see what the change does.

			choice
				`	"Have you asked someone senior for advice?"`
				`	"Don't worry about it."`
					goto worry
				`	"Can I help with something?"`
					goto help
			`	"I did. The senior researcher told me that most of the team feels it. There are some concrete theories, but nothing definitive, so they want me to investigate on my own without contaminating my thoughts with their ideas for now."`
				goto help
			label worry
			`	"Thanks for the advice, but that is our purpose here - to worry about all the stuff that humanity either cannot handle or that it turns a blind eye to. No, if there is something in the data to cause worry, we need to find it."`
			label help
			`	"If you would like to help, there is something you can do. I am currently putting together an expedition to Sagittarius A*, and I could use a transport..."`
			choice
				`	"I don't know where that is."`
					goto center
				`	"Isn't that the center of the galaxy?"`
					goto centeralt
				`	"I have been there."`

> 				`	"I have visited it a few times."`

Reviewer: Maybe an alternate negative response, like "You want me to go near a black hole?"

Author: Yeah, that works better than having two alternate "yep" options.

			`	"Yes, that is correct." She signs firmly.`
			label center
			`	Her signs take on a rhythmic pace as if reciting something. "Sagittarius A* is a strong astronomical radio source and a supermassive black hole that is located at the Center of the Milky Way. According to our best measurements, it has a mass of 3F6290 solar masses, and is surrounded by a ring of gas and dust that can range from 63 to 24D2 degrees Kelvin."`
			`	She returns to a conversational cadence. "Would you be interested in taking part in this expedition?"`
			choice
				`	"Sure, what do you need?"`
				`	"I'm busy right now, maybe later?"`
					defer
			`	"For this mission we will need some scientific equipment that is actually hardwired into your ship, not just carried along with us. When we arrive on <planet>, they will have the equipment ready to install. Just be aware that it will require 6 outfit space, and comes with a scientist, so you will need a free bunk for them, too.`
			`	"Once the laboratory module is installed, it will simply be a matter of travelling to Sagittarius A*. After our measurements, recordings, and observations are complete, we will return and report on our findings."`

> 			`	She glances down at her half-finished meal. "I am already packed and ready to go. I will meet you onboard the <ship> as soon as I am done eating."`

Reviewer: > "I am already packed and ready to go.

Reads weirdly. Perhaps: "I've already packed everything and am ready to go?"

Author: As a stylistic thing, the Remnant don't use contractions. Chalk it up to an idiosyncrasy in their language. But I'm open to adjusting the phrasing in other ways.

					goto labnext
				`	"Seeker?" you ask.`
			`	Dawn looks a little embarassed. "Seeker is a title of respect for deep space explorers. It is unofficial - just a token of respect - but it is both a title that I aspire to earn and one I do not feel that I deserve."`
			choice
				`	"Okay. So what next?"`
					goto labnext
				`	"I thought all Remnant were spacefarers?"`
			`	"Oh, we are. Every Remnant has had some experience in space. But there is a big difference between a rotation of a month or two of duty in a spacefaring post and spending months or years drifting in the void surrounding distant systems."`
			choice
				`	"Okay, let's get on with this."`
					goto labnext
				`	"Years drifting in the void?"`
			`	"If you want good data, you need to spend time to get it. And especially when we are tracking stellar phenomenon, you need periods of long observation. Snapshots of the sky are good - that is what we will be collecting on this trip. But for watching the galaxy at large, other civilizations, and things of that ilk... Nothing is better than long-term observation." She looks lost in thought for a moment.`
			label labnext
			branch jumpdrive

> 				has "Jump Drive" >= 1

Reviewer 2: This isn't a thing.

Author: So I need to have two separate missions in order to have this dialog; or just scrap the branch? (this is what the FW Pug 2 does - separate mission for each of Hyperdrive, Scram Drive, and Jump Drive)

Author: Removed this branch, as while it is useful and nice to be responsive to player's situation, was not necessary. Now Dawn doesn't know if you have a Jump Drive; and just tells you that it is needed for the next mission and how to get one, just in case you didn't know.

			`	"That is neither here nor there, however. Now that you have the research laboratory, you will need to locate a jump drive in order to reach Sagittarius A*." She pauses and checks her commlink. "That shouldn't be too hard. Our early warning system indicates that there are several incoming Korath fleets that should be here in the next day or two. Any one of them should have a Jump Drive you can take. Meet me in the Starport once you have a Jump Drive."`
			label jumpdrive
			`	""That is neither here nor there, however.." She pauses and checks her commlink. "I just need to take care of a few things, then I will meet you in the spaceport to embark to Sagittarius A*."`

event "remnant: lab availability"
	outfitter "Remnant"
		add "Research Laboratory"

mission "Remnant: Expanded Horizons Astral 2"
	name "Research expedition to Sagittarius A*"
	description "Travel to Sagittarius A* and then return to <destination>."
	blocked "You need at least 1 free bunk to take this mission."
	passengers 1
	waypoint "Sagittarius A*"
	to offer
		has "Remnant: Expanded Horizons Astral 1: done"
		require "Jump Drive" >= 1
		require "Research Laboratory" >= 1
	to complete

> 		require "Research Laboratory" >= 1

Reviewer 2: Also not a thing. #2494

Author: requirement for jump drive and research lab moved to `on offer` section.

			choice
				`	"So this is not the first expedition here?"`
				`	"Anything unusual in the data?"`
					goto astraldata
				`	"How much longer does the equipment need?"`
					goto astralfinish
			label astralfirst
			`	"No, this is the forty-third such expedition. Sagittarius A* was one of the first places our explorers sought to visit after we acquired Jump Drives, and we have been coming back regularly ever since. I know some of our reconnaissance pilots consider visiting here a pilgrimage of sorts. It is humbling to be here myself after reviewing so many data sets of past expeditions." She looks thoughtful, staring out the window at the the hub of the galaxy, as if her thoughts are jumping from idea to idea.`
			choice
				`	"Anything unusual in the data?"`
				`	"How much longer does the equipment need?"`
					goto astralfinish
			label astraldata

> 			`	"There are many oddities about this place. Every visit helps revise our physics models a little closer. But they also reveal problems." She frowns, her gestures taking on a curious tone. "For instance, you will note that your ramscoops are not picking up anything. According to standard astronomical theory, this system should be a veritable river of suitable material, and yet there is nothing. Not even the normal background trace amounts we find in deep space. We have visited other black holes, and they conform to the standard models in this respect. But not Sagittarius A*." She pauses, thinking about it for a moment.`

Reviewer: Ah yes, finally something on WHY THE HECK IS THERE NO FUEL WHEN CLOSE TO A BLACK HOLE.

Author: Sorry to not provide an answer to that question, but I figured it would be nice to throw it out there and acknowledge it, and point out that the Remnant are very curious about it.

			label astralmoving
			`	She gestures towards the airlock door. "Lead the way, Captain."`
				accept
	on stopover
		conversation
			`As the energy ripples from the Jump Drive fade, the spectacular void that is Sagittarius A* takes shape before you. You can make out a rough silhouette against the background of stars and the glow of the acretion disk. As you slowly fall towards the singularity, you activate the anti-gravity systems while various alarms start to trigger. While vastly underpowered for dealing with the immense pull of a black hole, they provide a degree of protection while you maintain a safe distance from it.`
			`	After a few minutes of just absorbing the spectacle before you, Dawn pulls herself away from the cockpit windows and scurries down to the lab. Leaving you to your thoughts, she quickly has the lab set up and running a variety of recordings, test, and observation equipment. After a few hours of work, she reappears on the bridge.`

> 			`	"Things are proceeding smoothly, Captain. All the sensors are working nominally and we are creating extensive archives of astronomical data as well. Another few hours and we should have all the data we need."`

Reviewer 3: Shouldn't it say normally here not nominaly, it seems and odd word choice
(sorry for comment in main discusion i am new to github)

Author: nominal is basically a technical synonym for normal. Very common in aerospace environments. (if you listen to SpaceX launches, you will hear it used in this way frequently.)

event "remnant: cognizance timer"
	system "Nenia"
		fleet "Ember Waste Node" 6000

mission "Remnant: Expanded Horizons Astral 1"
	name "Prep for an expedition to Sagittarius A*"
	description "To prepare for an expedition, you need to go to <destination> and have 6 space available for a new outfit."

> 	blocked "You need at least 1 free bunk to take this mission."

Author: Because this is a mission that involves ferrying a person around, there is a possibility that it is blocked because of lack of passenger space. I hope this message isn't invoked very often, but it is possible. 

It is also preventative for the second mission in the set, as it is made slightly more likely to have bunk problems because the outfit that is given in the `on complete` section actually requires 1 crew to operate. If the player is flying a penguin, it will mean that if they have *any* other passenger mission active, they will run into trouble in the next mission. So people that are hauling James or Katya or someone from one of the larger mission sets around for the fun of it are going to have to decide what they are going to do.

> 	blocked "You need at least 1 free bunk to take this mission."

Reviewer 4: But for a spaceport offered starting mission? Imo, if the player doesn't have the space, it's fine if they don't get offered it.

> 	blocked "You need at least 1 free bunk to take this mission."

Author: Sure, that makes sense.

	passengers 1
	minor
	to offer
		has "Remnant: Expanded Horizons Quarg 3"
		random < 25
	source
		government "Remnant"
	destination
		government "Remnant"
	on offer
		conversation
			`"In the starport cafeteria you come across Dawn sitting at a table looking worried. She smiles when she sees you, and waves you to the seat opposite her. "Back from galivanting around the galaxy with more tales to share?" she signs.`
			choice

> 				`	"Yes, as a matter of fact..."`

Author: I'm not sure this change fits. The phrase "Yes, as a matter of fact..." is meant to convey something akin to "Yes, actually...", with sort of a casual feel to it . Whereas "Yes, I have..." feels like it is more declarative, as if I am being decisive and pushy about it.

> 				`	"Yes, as a matter of fact..."`

Reviewer: If it were me, I would go for "As a matter of fact..." instead, since it's basically an affirmation.

> 				`	"Yes, as a matter of fact..."`

Author: OK, I'll be leaving this phrase as `	"Yes, as a matter of fact..."`

				`	"No, just hunting Korath..."`
				`	"No, just testing some stuff..."`
				`	"You look worried. Is something happening?"`
					goto worried
			`	She listens for a minute or two as you tell her about what you have been up to, but it is obvious that she has something on her mind.`
			choice
				`	"You look worried. Is everything alright?"`
					goto worried
				`	"You look distracted. We can talk later."`

> 					defer

Author: Personally, I'm the sort to go for the first response if someone looks concerned or preoccupied about something; but there are plenty of people who, upon seeing that someone is distracted and not paying attention, will just exit the situation. Could be they feel insulted that the person isn't paying them much attention, or it could be that they genuinely want to let the person focus on whatever has their attention. Either way, though, I think it is a valid option.

> 					defer

Reviewer: Suggestion: Have it be a question on whether the person wants some time. So: "You look distracted. Do you want me to come back later?"

> 					defer

Author: This defer has now been removed.

			label worried
			`	She makes a gesture of appreciation. "Yes, I'm worried, <first>. I keep feeling like there's something I'm missing." She pauses, then continues. "I should back up a step. While I do not have a permanent position yet, I am currently serving with an astrophysics team. We have been studying a variety of things, and I have been tasked with reviewing our bulk astrophysical data. It is fairly routine,  but I keep getting this prickly feeling that there's a problem that I am missing. Nothing obvious, of course. Our neural networks would pick up something obvious or repetitious really quickly. But the feeling just will not go away."`
			choice

> 				`	"Have you asked someone senior for advice?"`

Author: Hmmm, I'm not sure this fits. "A senior" suggests that "senior" is a classification or a rank, perhaps even a designated category or group of people. (For instance, in high school and some academic institutions, a Senior is a specific group within the student body.)

In this case, the choice is not supposed to be suggesting that she ask someone in a specific group, but rather just a general someone who happens to have more experience.

> 				`	"Have you asked someone senior for advice?"`

Author: Changed this line to "Have you asked someone with more experience for advice?" to be more clear about what they are asking.

			branch dawnsymbiont
				"remnant symbionts info" >= 1
			choice
				`	"OK, I will meet you onboard the ship."`
					accept
				`	"If you are full, you are full. Why push to eat more?"`
			branch dawnlicense
				has "license: Remnant Capital"
			`	She shrugs. "I burn a lot of calories, and eating healthy is important. Besides, I don't know how good food will be on your ship." She resumes eating with vigor and does not seem inclined to continue the conversation.`
				accept
			label dawnlicense
			`	"I am not yet used to the nutrient intake required to maintain my symbiont, so I have to consciously make sure that I eat enough to sustain it."`
			choice
				`	"Symbiont? What is that?"`
					goto whatis
				`	"You're just joking with me, forget I asked."`
			`	"Suit yourself. I will meet you onboard once I'm done." She shrugs, and returns to eating."`
				accept
			label whatis
			apply
				"remnant symbionts info" += 1
			`	"Oh, noone has told you about them? I thought once you'd demonstrated your trustworthiness and earned your Capital Ship License you would have found out." She pauses to take another bite, then continues.`
				goto dawndetails
			label dawnsymbiont
			choice

> 				`	"OK, I will meet you onboard the ship."`

Reviewer 4: ```suggestion
				`	"Okay, I will meet you onboard the ship."`
```

					accept

> 				`	"You are eating for two, too?"`

Reviewer 4: ```suggestion
				`	"You have one of those?"`
```

			`	She nods. "Yeah, I am not yet used to the nutrient intake required to maintain my symbiont, so I have to consciously make sure that I eat enough to sustain it."`
			choice

> 				`	"Interesting. Anyway, I will see you onboard the <ship>."`

Reviewer 4: ```suggestion
				`	"Interesting. I'll see you onboard the <ship>."`
```

					accept

> 				`	"I am curious to know more about them."`			

Reviewer 4: ```suggestion
				`	"Could you tell me more about them?"`			
```

			label dawndetails
			apply
				"remnant symbionts info" += 1
			`	"Basically, ages ago we domesticated some of the creatures found here, and discovered that they could be attached to us to form a symbiotic relationship." She pauses again to eat. "I only recently received mine, so I am not fully adjusted to it yet."`
			choice

> 				`	"OK, that's as much as I need to know."`

Reviewer 4: ```suggestion
				`	"Okay, that's as much as I need to know."`
```

				accept
			label onwards
			`	She nods. "Well, I will finish my meal and meet you on the <ship> shortly." With that, she returns to her meal.`
				accept
			label dawnwhere
			`	Dawn looks at you with an amused expression. "Just as every human is a unique individual, so every symbiont is also unique. As such, they attach to different areas of the body. Mine is located primarily along my spine." She twists around, and you can see a small layer of the tell-tale scaly leather extending up the back of her neck.`
			`	"We should probably be focusing on the expedition, though. You go get the <ship> ready for launch, and I will be there shortly."`
				accept
	on accept
		event "remnant: lab availability"
	on complete
		outfit "Research Laboratory"
		conversation
			`You follow the directions of the starport control and settle onto a pad next to the outfitter. You barely have a chance to finish your landing checks before a team of Remnant have a compact pod waiting on your loading ramp. Once you open the cargo bay doors they pick it up and maneuver it inside. They start securing it to the ship while one approaches you and Dawn.`

> 			`	"As requested, Dawn, we have loaded the research laboratory onboard. The team will install it if they can find the space, but if they do not, it will be secured in the cargo bay." The Remnant turns to you as she continues. "Before you depart, please verify that the laboratory is installed in the correct location. You know your ship better that we do, after all." She turns back to Dawn and makes a gesture of respect. "Embers guide your path, Seeker." With that, she strides away into the hive of activity that is the outfitting bays.`

Reviewer 4: ```suggestion
			`	"We have loaded the research laboratory onboard. The team will install it if they can find the space, but if they do not, it will be secured in the cargo bay." The Remnant turns to you as she continues. "Before you depart, please verify that the laboratory is installed in the correct location. You know your ship better that we do, after all." She turns back to Dawn and makes a gesture of respect. "Embers guide your path, Seeker." With that, she strides away into the hive of activity that is the outfitting bays.`
```

			choice
				`	"So, what is our next step?"`
					goto labnext
				`	"Seeker?" you ask.`
			`	Dawn looks a little embarassed. "Seeker is a title of respect for deep space explorers. It is unofficial - just a token of respect - but it is both a title that I aspire to earn and one I do not feel that I deserve."`
			choice

> 				`	"OK. So what next?"`

Reviewer 4: ```suggestion
				`	"Okay. So what next?"`
```

					goto labnext
				`	"I thought all Remnant were spacefarers?"`
			`	"Oh, we are. Every Remnant has had some experience in space. But there is a big difference between a rotation of a month or two of duty in a spacefaring post and spending months or years drifting in the void surrounding distant systems."`
			choice

> 				`	"OK. Let's get on with this."`

Reviewer 4: ```suggestion
				`	"Okay, let's get on with this."`
```

					goto labnext

> 				`	"Months or years drifing in the void?"`

Reviewer 4: ```suggestion
				`	"Years drifting in the void?"`
```

> 			`	"If you want good data, you need to spend time to get it. And especially when we are tracking stellar phenomenon, you need periods of long observation. Snapshots of the sky are good - that is what we will be colecting on this trip. But for watching the galaxy at large, other civilizations, and things of that ilk... Nothing is better than long-term observation." She looks lost in thought for a moment.`

Reviewer 4: ```suggestion
			`	"If you want good data, you need to spend time to get it. And especially when we are tracking stellar phenomenon, you need periods of long observation. Snapshots of the sky are good - that is what we will be collecting on this trip. But for watching the galaxy at large, other civilizations, and things of that ilk... Nothing is better than long-term observation." She looks lost in thought for a moment.`
```

			label labnext
			branch jumpdrive
				has "Jump Drive" >= 1
			`	"That is neither here nor there, however. Now that you have the research laboratory, you will need to locate a jump drive in order to reach Sagittarius A*." She pauses and checks her commlink. "That shouldn't be too hard. Our early warning system indicates that there are several incoming Korath fleets that should be here in the next day or two. Any one of them should have a Jump Drive you can take. Meet me in the Starport once you have a Jump Drive."`
			label jumpdrive

> 			`	""That is neither here nor there, however. Now that you have both the research laboratory and a jump drive, we will be able to reach Sagittarius A*." She pauses and checks her commlink. "I just need to take care of a few things, then I will meet you in the spaceport."`

Reviewer 4: ```suggestion
			`	""That is neither here nor there, however.." She pauses and checks her commlink. "I just need to take care of a few things, then I will meet you in the spaceport to embark to Sagittarius A*."`
```

	description "Travel to Sagittarius A* and then return to <destination>."
	blocked "You need at least 1 free bunk to take this mission."
	passengers 1
	waypoint "Sagittarius A*"
	to offer
		has "Remnant: Expanded Horizons Astral 1"
		require "Jump Drive" >= 1
		require "Research Laboratory" >= 1
	to complete
		require "Research Laboratory" >= 1
	to fail
		has "Research Laboratory" == 0
	source
		government "Remnant"
	destination
		government "Remnant"
	on offer
		conversation
			`Dawn is just finishing her meal when you enter the spaceport cafeteria. She stands up as you approach, and deposits the tray in a receptacle next to her table. "Well, ready to go?"`
			choice
				`	"Yes."`
					goto astralonwards
				`	"I should double check everything."`
			`	She nods. "I am ready, so I will come with you and wait onboard the <ship> while you do your checks."`
			label astralonwards
			`	She gestures towards the exit closest to your ship, and heads off at a quick pace. Outside, she moves calmly through the bustle of the starport, stepping around quick-moving freight movers and camel wagons alike. Arriving at the <ship>, she turns, and waits for you to catch up.`

> 			`	Just as you arrive and reach for the console to open the airlock, you sense a change in your surroundings. Looking around, you see that a group of Remnant gathered around the two of you, focused on Dawn. A young Remnant -scarecly more than a child - steps forward. "Thank you for your patience." He says, then steps back. An elderly Remnant steps forward. "Thank you for your curiosity." She says, then steps back. A middle-aged Remnant steps forward. "Go with our blessing, Seeker, and know that wherever you go, the Remnant go with you." She says it simply, but you feel the weight of tradition behind them. She turns to you. "And you, stranger who walks the stars... May the Embers guide your steps as you walk among us." She nods respectfully, then steps back.`

Reviewer 4: ```suggestion
			`	Just as you arrive and reach for the console to open the airlock, you sense a change in your surroundings. Looking around, you see that a group of Remnant gathered around the two of you, focused on Dawn. A young Remnant - scarcely more than a child - steps forward. "Thank you for your patience." He says, then steps back. An elderly Remnant steps forward. "Thank you for your curiosity." She says, then steps back. A middle-aged Remnant steps forward. "Go with our blessing, Seeker, and know that wherever you go, the Remnant go with you." She says it simply, but you feel the weight of tradition behind them. She turns to you. "And you, stranger who walks the stars... May the Embers guide your steps as you walk among us." She nods respectfully, then steps back.`
```

			`	"Long ago we, as a people, decided that we must face the fear of separation, the unknown, and the danger. One of the ways we do that is by holding these informal ceremonies. When one of us is going to depart on their first long distance voyage into hostile territory, a group of Remnant gather to bid them farewell. It is a way to strengthen ourselves and remind each other of our connections."`
			choice
				`	"Thanks for the explanation."`
					goto astralmoving
				`	"Kind of like a funeral, with friends and family gathering to see you off?"`
			`	"Oh, no. I said my farewells to my friends and relatives before we left. Those who gathered to see me off today were more of a representation of Remnant society. I only recognized a few of them, but they took the time to come down to the <ship> to see me before I left. They came to see me, and wish me well, so that no matter what challenges I face, I will know that the Remnant remember me and appreicate what I am doing." Her gestures fade into contemplative stillness for a moment, then resume motion.`
			label astralmoving
			`	She gestures towards the airlock door. "Lead the way, Captain."`
				accept
	on stopover
		conversation

> 			`As the energy ripples from the Jump Drive fade, the spectacular void that is Sagittarius A* takes shape before you. You can make out a rough silhouette against the background of stars and the glow of the acretion disk. As various alarms start to trigger you reach over and activate the antigravity systems. While vastly underpowered for dealing with the immense pull of a black hole, they provide a degree of protection while you maintain a safe distance from it.`

Reviewer 4: ```suggestion
			`As the energy ripples from the Jump Drive fade, the spectacular void that is Sagittarius A* takes shape before you. You can make out a rough silhouette against the background of stars and the glow of the acretion disk. As you slowly fall towards the singularity, you activate the anti-gravity systems while various alarms start to trigger. While vastly underpowered for dealing with the immense pull of a black hole, they provide a degree of protection while you maintain a safe distance from it.`
```

			`	"I will be going outside for a few minutes." Dawn declares. "I shall just be on the hull."`
			choice
				`	"Let me know when you are done."`
					goto astraldone
				`	"Would you mind if I came too?"`
			`	"Not at all, Captain. The view should be remarkable, and so long as we stay inside the shields, the radiation will not pose a problem." You quickly don your spacesuit, and join Dawn outside on the hull. As promised, the view is indeed magnificent. Dawn signs ocassionally, pointing out various features, and you appreciate that out here, sound would just be incongruous with the natural splendor.`
			label astraldone
			`	After what feels like hours the equipment notifies you that it has finished taking its readings, and Dawn packs up the data in carefully shielded data crystals. Everything is ready to return to <destination>.`
	on complete
		payment 500000
		conversation

> 			`The journey back is a blur as the <ship> jumps through countless systems. Gradually the brilliant density of stars that make up the inner core fades into the ominous and yet comfortingly familiar red glow of the Ember Waste. Finally you slide into the approach patterns of <destination>, quickly tapping out your code into the IFF system and watching as the protective gaze of the anti-air emplacements sweep the skies around you for shadowing craft.`

Reviewer 4: ```suggestion
			`The journey back to <planet> was blur as the <ship> jumped through countless systems. Gradually the brilliant density of stars that make up the inner core faded into the ominous and yet comfortingly familiar red glow of the Ember Waste. Finally you slide into the approach patterns of <planet>, quickly tapping out your code into the IFF system and watching as the protective gaze of the anti-air emplacements sweep the skies around you for shadowing craft.`
```

> 			`You settle onto your assigned pad, and within seconds Remnant technicians have the resupply cables hooked up, while a cheerful group greet Dawn and help her unload her luggage and data crystals. "Well, <first>, it has been a good trip." She taps at her commlink. I have transfered our expedition's logistics alotment of <payment> to you. And in case you are interested, you are now eligible to do the regular data update missions, which will be located in the job board. We always need fresh data, after all." She nods her head in respect to you. "Thank you, and may the Embers watch over you." She turns and heads off into the flows of people.`

Reviewer 4: ```suggestion
			`You settle onto your assigned pad, and within seconds Remnant technicians have resupply cables hooked up, while a cheerful group greet Dawn and help her unload her luggage and data crystals. "Well, <first>, it has been a good trip." She taps at her commlink. I have transferred our expedition's logistics allotment of <payment> to you. And in case you're wondering what to do with your Research Laboratory, you can keep it. We always need fresh data, and if you are interested, you can check up on places designated in the job board." She nods her head in respect to you. "Thank you, and may the Embers watch over you." She turns and heads off into the flows of people.`
```

mission "Remnant: Expanded Horizons Astral job"
	name "Research Expedition to <waypoint>."

> 	description "Travel to <waypoint> with a research laboratory and then return to <destination>."

Reviewer 4: ```suggestion
	description "Travel to <waypoint> with a research laboratory to collect data, then return to <destination>."
```

	job
	repeat
	waypoint
		not attributes "pleiades"
	to offer
		has "Remnant: Expanded Horizons Astral 2: done"
		require "Research Laboratory" >= 1
	to complete
		require "Research Laboratory" >= 1
	to fail
		has "Research Laboratory" == 0
	source
		government "Remnant"

> 	destination
> 		government "Remnant"

Reviewer 4: Just have the mission return to its source.
```suggestion
```

> 	on offer
> 		dialog "Please take a research laboratory to <waypoint> to collect fresh data"

Reviewer 4: ~smh zitchas jobs should've have on offers~
```suggestion
```

	on complete
		payment 400000
		"reputation: Remnant" ++

> 		dialog "A researcher quickly retrieves the data crystals and hands you the logistics allotment of <payment>."

Reviewer 4: ```suggestion
		dialog "When you land, a researcher quickly retrieves the data crystals with the information about the surveyed system and hands you the logistics allotment of <payment>."
```

event "remnant: cognizance timer"
	system "Nenia"
		fleet "Ember Waste Node" 6000

mission "Remnant: Expanded Horizons Astral 1"
	name "Preparation for expedition"
	description "To prepare for an expedition to Sagittarius A*, go to <destination> with 6 outfit space available."
	passengers 1
	minor
	to offer
		has "Remnant: Expanded Horizons Quarg 3: done"
		# random < 25
	source
		government "Remnant"
	destination

> 		government "Remnant"

Author: Yes. I've just commented out the random to make it easier to test.

	on offer
		conversation
			`"In the starport cafeteria you come across Dawn sitting at a table looking worried. She smiles when she sees you, and waves you to the seat opposite her. "Back from gallivanting around the galaxy with more tales to share?" she signs.`
			choice
				`	"Yes, as a matter of fact..."`
				`	"No, just hunting Korath..."`
				`	"No, just testing some stuff..."`
				`	"You look worried. Is something happening?"`
					goto worried
			`	She listens for a minute or two as you tell her about what you have been up to, but it is obvious that she has something on her mind.`
			choice
				`	"You look worried. Is everything alright?"`
					goto worried

> 				`	"You look distracted. We can talk later."`

Author: OK, changed things up a bit. That conversation became a simple interaction conversation, and moved the option to back out to the following conversation. 

And yeah, backing out of the conversation is a callous move.

The option to decline the mission has been added to the conversation after Dawn mentions what the mission is.

			`	She returns to a conversational cadence. "Would you be interested in taking part in this expedition?"`
			choice
				`	"Sure, what do you need?"`
				`	"I'm busy right now, maybe later?"`
					defer
			`	"For this mission we will need some scientific equipment that is actually hardwired into your ship, not just carried along with us. When we arrive on <planet>, they will have the equipment ready to install. Just be aware that it will require 6 outfit space, and comes with a scientist, so you will need a free bunk for them, too.`
			`	"Once the laboratory module is installed, it will simply be a matter of travelling to Sagittarius A*. After our measurements, recordings, and observations are complete, we will return and report on our findings."`
			`	She glances down at her half-finished meal. "I am already packed and ready to go. I will meet you onboard the <ship> as soon as I am done eating."`
			branch dawnsymbiont

> 				"remnant symbionts info" >= 1

Reviewer 2: ```suggestion
				has "remnant symbionts info"
```

				accept
	on accept
		event "remnant: lab availability"
	on complete
		outfit "Research Laboratory"
		conversation
			`You follow the directions of the starport control and settle onto a pad next to the outfitter. You barely have a chance to finish your landing checks before a team of Remnant have a compact pod waiting on your loading ramp. Once you open the cargo bay doors they pick it up and maneuver it inside. They start securing it to the ship while one approaches you and Dawn.`
			`	"We have loaded the research laboratory onboard. The team will install it if they can find the space, but if they do not, it will be secured in the cargo bay." The Remnant turns to you as she continues. "Before you depart, please verify that the laboratory is installed in the correct location. You know your ship better that we do, after all." She turns back to Dawn and makes a gesture of respect. "Embers guide your path, Seeker." With that, she strides away into the hive of activity that is the outfitting bays.`
			choice
				`	"So, what is our next step?"`
					goto labnext

> 				`	"Seeker?" you ask.`

Reviewer 2: ```suggestion
				`	"Seeker?"`
```

			`	"If you want good data, you need to spend time to get it. And especially when we are tracking stellar phenomenon, you need periods of long observation. Snapshots of the sky are good - that is what we will be collecting on this trip. But for watching the galaxy at large, other civilizations, and things of that ilk... Nothing is better than long-term observation." She looks lost in thought for a moment.`
			label labnext
			`	"That is neither here nor there, however. Now that you have the research laboratory, you will need to locate a jump drive in order to reach Sagittarius A*." She pauses and checks her commlink. "If you do not already have one, you can salvage one from a Korath Raider. Our early warning system indicates that there are several incoming Korath fleets that should be here in the next day or two. Any one of them should have a Jump Drive you can take. Meet me in the Starport once you have a Jump Drive."`

event "remnant: lab availability"
	outfitter "Remnant"
		add "Research Laboratory"

mission "Remnant: Expanded Horizons Astral 2"
	name "Research expedition to Sagittarius A*"
	description "Travel to Sagittarius A* and then return to <destination>."

> 	blocked "You need at least 1 free bunk to take this mission."

Reviewer 2: This could also be blocked by not having a JD or lab.

Author: Fixed. Thanks!

	passengers 1
	waypoint "Sagittarius A*"

> 	to offer
> 		has "Remnant: Expanded Horizons Astral 1: done"
> 	source
> 		government "Remnant"
> 	destination
> 		government "Remnant"

Reviewer 2: ```suggestion
	source
		government "Remnant"
	destination
		government "Remnant"
	to offer
		has "Remnant: Expanded Horizons Astral 1: done"
```

	on offer

> 		require "Jump Drive" 1
> 		require "Research Laboratory" 1

Reviewer 2: `require 1` is the same as just `require`, so might as well do that.
```suggestion
		require "Jump Drive"
		require "Research Laboratory"
```

			`The journey back to <planet> was blur as the <ship> jumped through countless systems. Gradually the brilliant density of stars that make up the inner core faded into the ominous and yet comfortingly familiar red glow of the Ember Waste. Finally you slide into the approach patterns of <planet>, quickly tapping out your code into the IFF system and watching as the protective gaze of the anti-air emplacements sweep the skies around you for shadowing craft.`
			`You settle onto your assigned pad, and within seconds Remnant technicians have resupply cables hooked up, while a cheerful group greet Dawn and help her unload her luggage and data crystals. "Well, <first>, it has been a good trip." She taps at her commlink. I have transferred our expedition's logistics allotment of <payment> to you. And in case you're wondering what to do with your Research Laboratory, you can keep it. We always need fresh data, and if you are interested, you can check up on places designated in the job board." She nods her head in respect to you. "Thank you, and may the Embers watch over you." She turns and heads off into the flows of people.`

mission "Remnant: Expanded Horizons Astral job"

> 	name "Research Expedition to <waypoints>."

Reviewer 2: Period in a name.
```suggestion
	name "Research Expedition to <waypoints>"
```

	description "Travel to <waypoints> with a research laboratory to collect data, then return to <destination>."
	job
	repeat
	waypoint
		not attributes "pleiades"
	to offer
		has "Remnant: Expanded Horizons Astral 2: done"
	on offer

> 		require "Research Laboratory" 1

Reviewer 2: ```suggestion
		require "Research Laboratory"
```

event "remnant: cognizance timer"
	system "Nenia"
		fleet "Ember Waste Node" 6000

mission "Remnant: Expanded Horizons Astral 1"
	name "Preparation for expedition"
	description "To prepare for an expedition to Sagittarius A*, go to <destination> with 6 outfit space available."
	passengers 1
	minor
	source
		government "Remnant"
	destination
		government "Remnant"
	to offer
		has "Remnant: Expanded Horizons Quarg 3: done"

> 		# random < 25

Author: OK. Removed the commenting.

	on offer
		conversation
			`In the starport cafeteria you come across Dawn sitting at a table looking worried. She smiles when she sees you, and waves you to the seat opposite her. "Back from gallivanting around the galaxy with more tales to share?" she signs.`
			choice
				`	"Yes, as a matter of fact..."`
				`	"No, just hunting Korath..."`
				`	"No, just testing some stuff..."`
				`	"You look worried. Is something happening?"`
					goto worried
			`	She listens for a minute or two as you tell her about what you have been up to, but it is obvious that she has something on her mind.`
			choice
				`	"You look worried. Is everything alright?"`
				`	"You look distracted. Do you want to talk about it?"`
			label worried

> 			`	She makes a gesture of appreciation. "Yes, I'm worried, <first>. I keep feeling like there's something I'm missing." She frowns, contemplating something unseen. "I should back up a step. While I do not have a permanent position yet, I am currently serving with an astrophysics team. We have been studying a variety of things, and I have been tasked with reviewing our bulk astrophysical data. It is fairly routine,  but I keep getting this prickly feeling that there's a problem that I am missing. Nothing obvious, of course. Our neural networks would pick up something obvious or repetitious really quickly. But the feeling just will not go away."`

Author: `	She makes a gesture of appreciation. "Yes, I'm worried, <first>. I keep feeling like there's something I'm missing." She frowns, contemplating something unseen. "I should back up a step. Since we last talked, I was assigned to serve with an astrophysics team. My current work is part of the quantum condensate fabric density project, and I have been tasked with reviewing our bulk astrophysical data. It is fairly routine,  but I keep getting this prickly feeling that there's a problem that I am missing. Nothing obvious, of course. Our neural networks would pick up something obvious or repetitious really quickly. But the feeling just will not go away."`

OK, revised the paragraph to remove some of the ambiguity and clarify a few things. 

Amusingly, I've had very similar conversations with people, where I was in the same position as <first>. Is it frustrating? Yes. Is it confusing? Yes. Is it realistic and not conveniently parsed for optimum understanding? Yes. Is it the intended effect? Yes. At least to start with, anyway.  The intent was that the first bit of the paragraph is frustrating, like it usually is when one starts talking to someone who is several hours into a complex train of thought. The remainder is supposed to provide some background and clarify things a bit.

> 

> > I keep feeling like there's something I'm missing.
> 
> Missing from what? What are we talking about?

The astrophysical data, presumably. But she doesn't know if it is a problem with the data, or some sort of problem revealed by the data. She just knows she's picking up on something not jiving the way she thinks it should.

> > I do not have a permanent position yet...
> 
> A permanent position for what?

A career, a job, a work placement, somewhere to work proffessionally...
 
> > serving with an astrophysics teams
> 
> What are they doing?
> 
> > studying a variety of things
> 
> What things?

the quantum condensate fabric density project

> > prickling feeling that there's a problem that I am missing
> 
> Any ideas?

I'm sorry, I'm not an applied astrophysicist from the 31st century, I don't know exactly what sort of thing would be considered cutting edge research. I'm trying to avoid techno-babble where possible, but I guess this is one place where some is necessary. 

> > The senior researcher told me that most of the team feels it. There are some concrete theories, but nothing definitive
> 
> Feel what? Theories on what?
> 
> > Being able to take part in these expeditions is something I have been looking forward to for ages
> 
> What expeditions?
> 
> > I am currently putting together an expedition to Sagittarius A*
> 
> Why?

`to collect some fresh data I can use to check our records` (added text to explain this)

> 			`	She makes a gesture of appreciation. "Yes, I'm worried, <first>. I keep feeling like there's something I'm missing." She frowns, contemplating something unseen. "I should back up a step. While I do not have a permanent position yet, I am currently serving with an astrophysics team. We have been studying a variety of things, and I have been tasked with reviewing our bulk astrophysical data. It is fairly routine,  but I keep getting this prickly feeling that there's a problem that I am missing. Nothing obvious, of course. Our neural networks would pick up something obvious or repetitious really quickly. But the feeling just will not go away."`

Reviewer 2: If the intention is that it's frustrating then I'd expect a player choice to express that.

> 			`	She makes a gesture of appreciation. "Yes, I'm worried, <first>. I keep feeling like there's something I'm missing." She frowns, contemplating something unseen. "I should back up a step. While I do not have a permanent position yet, I am currently serving with an astrophysics team. We have been studying a variety of things, and I have been tasked with reviewing our bulk astrophysical data. It is fairly routine,  but I keep getting this prickly feeling that there's a problem that I am missing. Nothing obvious, of course. Our neural networks would pick up something obvious or repetitious really quickly. But the feeling just will not go away."`

Author: Before I start doing that, do the changes help alleviate enough of the potential ambiguity? There's a fair bit more detail than there used to be.

			choice
				`	"Have you asked someone with more experience for advice?"`
				`	"Don't worry about it."`
					goto worry
				`	"Can I help with something?"`
					goto help
				`	"I have to go. I will stop by to talk more later."`

> 					defer

Author: That's true. Move the defer to the start, along the lines of what you suggested here.

			`	"I did. The senior researcher told me that most of the team feels it. There are some concrete theories, but nothing definitive, so they want me to investigate on my own without contaminating my thoughts with their ideas for now."`
				goto help
			label worry
			`	"Thanks for the advice, but that is our purpose here - to worry about all the stuff that humanity either cannot handle or that it turns a blind eye to. No, if there is something in the data to cause worry, we need to find it."`
			label help
			`	"If you would like to help, there is something you can do. I am currently putting together an expedition to Sagittarius A*, and I could use a transport..."`
			choice
				`	"I don't know where that is."`
					goto center
				`	"Isn't that the center of the galaxy?"`
					goto centeralt
				`	"You want me to go near a blackhole?!"`
					goto centeralt
				`	"I have been there."`
				`	"Sorry, I'm not interested in visiting a blackhole."`

> 					decline

Author: This conversation is now down to three options, and the decline has been moved to the next conversational choice.

			`	"Really?" She looks surprised. "I have seen sensor logs and recordings from previous visits, but not many people have actually been there." She pauses to take a bite from her neglected meal. "Being able to take part in these expeditions is something I have been looking forward to for ages, so of course Sagittarius A* is the first place on my list."`
				goto center
			label centeralt
			`	"Yes, that is correct." She signs firmly.`
			label center
			`	Her signs take on a rhythmic pace as if reciting something. "Sagittarius A* is a strong astronomical radio source and a supermassive black hole that is located at the Center of the Milky Way. According to our best measurements, it has a mass of 3F6290 solar masses, and is surrounded by a ring of gas and dust that can range from 63 to 24D2 degrees Kelvin."`
			`	She returns to a conversational cadence. "Would you be interested in taking part in this expedition?"`
			choice
				`	"Sure, what do you need?"`
				`	"I'm busy right now, maybe later?"`

> 					defer

Author: Agreed. Moved the "I'm not interested" choice to here in place of the defer.

			`	"For this mission we will need some scientific equipment that is actually hardwired into your ship, not just carried along with us. When we arrive on <planet>, they will have the equipment ready to install. Just be aware that it will require 6 outfit space, and comes with a scientist, so you will need a free bunk for them, too.`
			`	"Once the laboratory module is installed, it will simply be a matter of travelling to Sagittarius A*. After our measurements, recordings, and observations are complete, we will return and report on our findings."`
			`	She glances down at her half-finished meal. "I am already packed and ready to go. I will meet you onboard the <ship> as soon as I am done eating."`
			branch dawnsymbiont
				has "remnant symbionts info"
			choice
				`	"Okay, I will meet you onboard the ship."`
					accept

> 				`	"Why do you need to finish your food?"`

Author: That makes sense. changed it a litle bit to: `	"Developed quite the appetite, haven't you?"` Also replaced the equivalent line in the other branch with the same question, so both branches start off the same and the changes really start with <first>'s response to her comment about the symbiont.

			choice
				`	"Interesting. I'll see you onboard the <ship>."`
					accept
				`	"Could you tell me more about them?"`			
			label dawndetails
			apply
				"remnant symbionts info" += 1
			`	"Basically, ages ago we domesticated some of the creatures found here, and discovered that they could be attached to us to form a symbiotic relationship." She pauses again to eat. "I only recently received mine, so I am not fully adjusted to it yet."`
			choice
				`	"Okay, that's as much as I need to know."`
					goto onwards
				`	"What is it like to have a symbiont?"`
			`	"Oh, it is wonderful. It is very hard to describe. At first it is just slimy, then a lot of tingling. The tingling lasted for several hours, and then it was just like waking up: Rising up out of darkness to a brighter world with just so much to see and learn. Suddenly, it was as if everything was moving slower and I could take a long hard look at everything..." Dawn is visibly excited, signing so fast you have trouble following.`
			choice
				`	"Thanks for the description. I should go prep the <ship> now."`
					accept
				`	"Where is your symbiont? Yours does not seem to be obvious."`
					goto dawnwhere

> 				`	"Could I see your symbiont?"`

Author: I was running with a variation on the "If someone says they have a tatoo somewhere not visible, you've got three choices: 
1) Move on and talk about something else.
2) Ask what the tatoo is (looking for a description, and most people are comfortable describing it).
3) Ask to see it. (which, if it is somewhere that requires removing clothing to show, is something a lot of people are going to be uncomfortable with.)

So basically, I was trying to give people the option to ask about it politely, or impolitely.

> 				`	"Could I see your symbiont? The ones I have seen are visible, like gloves or neck braces."`

Author: Yeah, that bit of extra dialog would only fit if the person has seen them before. Removed the second sentence, as the first part is a valid question regardless of situation. So now that question is quite simply:
`	"Could I see your symbiont?"`

			choice
				`	"So, what is our next step?"`
					goto labnext
				`	"Seeker?"`
			`	Dawn looks a little embarassed. "Seeker is a title of respect for deep space explorers. It is unofficial - just a token of respect - but it is both a title that I aspire to earn and one I do not feel that I deserve."`
			choice
				`	"Okay. So what next?"`
					goto labnext
				`	"I thought all Remnant were spacefarers?"`
			`	"Oh, we are. Every Remnant has had some experience in space. But there is a big difference between a rotation of a month or two of duty in a spacefaring post and spending months or years drifting in the void surrounding distant systems."`
			choice
				`	"Okay, let's get on with this."`
					goto labnext
				`	"Years drifting in the void?"`
			`	"If you want good data, you need to spend time to get it. And especially when we are tracking stellar phenomenon, you need periods of long observation. Snapshots of the sky are good - that is what we will be collecting on this trip. But for watching the galaxy at large, other civilizations, and things of that ilk... Nothing is better than long-term observation." She looks lost in thought for a moment.`
			label labnext

> 			`	"That is neither here nor there, however. Now that you have the research laboratory, you will need to locate a jump drive in order to reach Sagittarius A*." She pauses and checks her commlink. "If you do not already have one, you can salvage one from a Korath Raider. Our early warning system indicates that there are several incoming Korath fleets that should be here in the next day or two. Any one of them should have a Jump Drive you can take. Meet me in the Starport once you have a Jump Drive."`

Author: Are you referring to the player's question of "Years drifting in the void?" leading to "If you want good data, you need to spend time getting it?"

This is intended to be Dawn skipping the "short answer" part of the conversation to go straight to the "here's the larger situation and explanation" part. It isn't the best conversational practice; but it is one that is common in technical and academic environments, at least in my experience.

> 			`	"That is neither here nor there, however. Now that you have the research laboratory, you will need to locate a jump drive in order to reach Sagittarius A*." She pauses and checks her commlink. "If you do not already have one, you can salvage one from a Korath Raider. Our early warning system indicates that there are several incoming Korath fleets that should be here in the next day or two. Any one of them should have a Jump Drive you can take. Meet me in the Starport once you have a Jump Drive."`

Reviewer 2: "So, what is our next step?" was leading into "That is neither here nor there, however."

event "remnant: lab availability"
	outfitter "Remnant"
		add "Research Laboratory"

mission "Remnant: Expanded Horizons Astral 2"
	name "Research expedition to Sagittarius A*"
	description "Travel to Sagittarius A* and then return to <destination>."
	blocked "You need at least 1 free bunk, a jump drive, and a research laboratory to take this mission."
	passengers 1
	waypoint "Sagittarius A*"
	source
		government "Remnant"
	destination
		government "Remnant"
	to offer
		has "Remnant: Expanded Horizons Astral 1: done"
	on offer
		require "Jump Drive"
		require "Research Laboratory"
		conversation
			`Dawn is just finishing her meal when you enter the spaceport cafeteria. She stands up as you approach, and deposits the tray in a receptacle next to her table. "Well, ready to go?"`
			choice
				`	"Yes."`
					goto astralonwards
				`	"I should double check everything."`

> 			`	She nods. "I am ready, so I will come with you and wait onboard the <ship> while you do your checks."`

Author: Added the `defer`.

			`	"Long ago we, as a people, decided that we must face the fear of separation, the unknown, and the danger. One of the ways we do that is by holding these informal ceremonies. When one of us is going to depart on their first long distance voyage into hostile territory, a group of Remnant gather to bid them farewell. It is a way to strengthen ourselves and remind each other of our connections."`
			choice
				`	"Thanks for the explanation."`
					goto astralmoving
				`	"Kind of like a funeral, with friends and family gathering to see you off?"`
			`	"Oh, no. I said my farewells to my friends and relatives before we left. Those who gathered to see me off today were more of a representation of Remnant society. I only recognized a few of them, but they took the time to come down to the <ship> to see me before I left. They came to see me, and wish me well, so that no matter what challenges I face, I will know that the Remnant remember me and appreicate what I am doing." Her gestures fade into contemplative stillness for a moment, then resume motion.`
			label astralmoving
			`	She gestures towards the airlock door. "Lead the way, Captain."`
				accept

> 	on enter "Sagittarius A*"

Author: Is it possible to specifyconditions for this, like a `to waypoint` in the same way that we can specify conditions for, say, an `on complete` ? Specifically, asking if it is possible to check here to ensure that the player does actually have the "Research Laboratory" on their ship, and didn't just ditch it after they accepted the mission?

(I realize I could specify taking off the "Research Laboratory" to be a fail condition, but I don't want to hinder people's outfitting or fail the mission - I just don't want them to be able to complete the waypoint without it.)

> 	on enter "Sagittarius A*"

Reviewer 2: `require` still goes in `on *` blocks, so theoretically one could prevent the `on waypoint` action from triggering if one didn't have a specific outfit, *but* the waypoint objective would still be complete for that system, meaning that you can prevent the `on waypoint` from showing, but if it doesn't show and the player returns with the correct outfits, it won't then trigger the `on waypoint` because the waypoint objective was already complete.

> 	on enter "Sagittarius A*"

Author: hmmm... That's problematic.

> 	on enter "Sagittarius A*"

Reviewer 2: Just don't worry about it. The mission requires that the player have one at the start. If they're really going to go through the motions of having the outfit installed in order to start the mission then removing it after accepting the mission, then let them have at it. We can't stop them from doing that, so it's on them that the story doesn't make sense without a lab installed.

> 	on enter "Sagittarius A*"

Reviewer 2: If you *really* wanted to require that the player have a lab installed, try putting `require "Research Laboratory"` in the `on enter`, then have the `on enter` also set a condition that is required for mission completion. This way the `on enter` won't trigger unless the lab is present, and the mission can't be completed unless the `on enter` is triggered. What you couldn't do though is have a dialog pop up if the player arrives without a lab.

> 	on enter "Sagittarius A*"

Author: That's true. I'll just leave this as is. If the player *really* wants to jump through hoops to do the mission without a lab...

		conversation

> 			`As the energy ripples from the Jump Drive fade, the spectacular void that is Sagittarius A* takes shape before you. You can make out a rough silhouette against the background of stars and the glow of the acretion disk. As you slowly fall towards the singularity, you activate the anti-gravity systems while various alarms start to trigger. While vastly underpowered for dealing with the immense pull of a black hole, they provide a degree of protection so long as you maintain a safe distance from it.`

Reviewer 5: Consider the distance of our sun to alpha centauri. That is more than a Parsec away and there's 1 star in that local distance. This close to the blackhole, even in a galaxy much smaller than our own, there would be 500 stars, of average mass 50-500 times greater than our own sun.

Basically, seeing this against the background of stars in this area is like looking at a brilliant halo around a dark spot while staring at the sky in broad daylight with 12,000 suns equivalent scattered about the sky.

Space is not dark here. The only thing dark is the black hole itself. Everything else is going to be eye-wateringly bright. At the very least there should be mention of some *really* heavy duty filters being placed on whatever view you're using.

Author: Thanks for the astronomical input. I've updated the description in this section to:

`As the energy ripples from the Jump Drive fade, the spectacular void that is Sagittarius A* takes shape before you. Compared to the vast darkness of space that you are accustomed to, reality here feels almost like a color negative: Even with the automatic filtering in your windows, space is incredibly bright with the densely packed stars that provide a backdrop for Sagittarius A*. The moment of awe is shattered by your alarms warning you of your impending doom as the <ship> begins to fall inwards towards the black hole. You quickly activate your anti-gravity system, which, although vastly under-powered for dealing with the immense pull of a black hole, does provide a degree of protection so long as you maintain a safe distance from it.`

Does that work better?

Reviewer 5: That works, at least for the level of implemented mechanics thus far.

> 			`As the energy ripples from the Jump Drive fade, the spectacular void that is Sagittarius A* takes shape before you. Compared to the vast darkness of space that you are accustomed to, reality here feels almost like a color negative: Even with the automatic filtering in your windows, space is incredibly bright with the densely packed stars that provide a backdrop for Sagittarius A*. The moment of awe is shattered by your alarms warning you of your impending doom as the <ship> begins to fall inwards towards the black hole. You quickly activate your anti-gravity system, which, although vastly under-powered for dealing with the immense pull of a black hole, does provide a degree of protection so long as you maintain a safe distance from it.`

Reviewer 2: ```suggestion
			`As the energy ripples from the jump drive fade, the spectacular void that is Sagittarius A* takes shape before you. Compared to the vast darkness of space that you are accustomed to, reality here feels almost like a color negative: even with the automatic filtering in your windows, space is incredibly bright with the densely packed stars that provide a backdrop for Sagittarius A*. The moment of awe is shattered by your alarms warning you of your impending doom as the <ship> begins to fall inwards towards the black hole. You quickly activate your repulsor engines, which, although vastly under-powered for dealing with the immense pull of a black hole, does provide a degree of protection so long as you maintain a safe distance from it.`
```

			`	After a few minutes of just absorbing the spectacle before you, Dawn pulls herself away from the cockpit windows and scurries down to the lab. Leaving you to your thoughts, she quickly has the lab set up and running a variety of recordings, test, and observation equipment. After a few hours of work, she reappears on the bridge.`
			`	"Things are proceeding smoothly, Captain. All the sensors are working nominally and we are creating extensive archives of astronomical data as well. Another few hours and we should have all the data we need."`
			choice
				`	"Good work. What kind of data are you recording?`
				`	"Sounds good. Just let me know when you are ready to leave."`
					goto astraldone

> 			`	She smiles, and replies. "It might be easier to ask what data we are leaving out. The data we are collecting broadly falls into three data sets: Data on Sagittarius A* itself, data on its environment, and observations of the surrounding galaxy from here. Obviously, we cannot go take samples of Sagittarius A* itself, nor take direct measurements of most of its properties, but we can get data from increasingly minute distances from the event horizon. We can then add this data to that gathered by previous expeditions, note the differences, and use it in our modeling."`

Reviewer 2: ```suggestion
			`	She smiles, and replies. "It might be easier to ask what data we are leaving out. The data we are collecting broadly falls into three data sets: data on Sagittarius A* itself, data on its environment, and observations of the surrounding galaxy from here. Obviously, we cannot go take samples of Sagittarius A* itself, nor take direct measurements of most of its properties, but we can get data from increasingly minute distances from the event horizon. We can then add this data to that gathered by previous expeditions, note the differences, and use it in our modeling."`
```

			choice
				`	"So this is not the first expedition here?"`
				`	"Anything unusual in the data?"`
					goto astraldata
				`	"How much longer does the equipment need?"`
					goto astralfinish
			label astralfirst

> 			`	"No, this is the forty-third such expedition. Sagittarius A* was one of the first places our explorers sought to visit after we acquired Jump Drives, and we have been coming back regularly ever since. I know some of our reconnaissance pilots consider visiting here a pilgrimage of sorts. It is humbling to be here myself after reviewing so many data sets of past expeditions." She looks thoughtful, staring out the window at the the hub of the galaxy, as if her thoughts are jumping from idea to idea.`

Reviewer 2: ```suggestion
			`	"No, this is the forty-third such expedition. Sagittarius A* was one of the first places our explorers sought to visit after we acquired jump drives, and we have been coming back regularly ever since. I know some of our reconnaissance pilots consider visiting here a pilgrimage of sorts. It is humbling to be here myself after reviewing so many data sets of past expeditions." She looks thoughtful, staring out the window at the the hub of the galaxy, as if her thoughts are jumping from idea to idea.`
```

			choice
				`	"Anything unusual in the data?"`
				`	"How much longer does the equipment need?"`
					goto astralfinish
			label astraldata

> 			`	"There are many oddities about this place. Every visit helps revise our physics models a little closer. But they also reveal problems." She frowns, her gestures taking on a curious tone. "For instance, you will note that your ramscoops are not picking up anything. According to standard astronomical theory, this system should be a veritable river of suitable material, and yet there is nothing. Not even the normal background trace amounts we find in deep space. We have visited other black holes, and they conform to the standard models in this respect. But not Sagittarius A*." She pauses, thinking about it for a moment.`

Reviewer 5: So we are canonising this oddity then? I wonder, could it be worthwhile to consider the possibility of having another black hole system other than Sag A that could be acting strangely?

In this location, not only would the black hole be acting strangely to not have solar wind, but whatever it was doing would have to be strong enough to counteract 12,000 suns worth of solar wind and keep them away from it.

Author: Yes. All is not right in the state of Sagittarius A*. And it is far more than just a quirk in the standard model, too.

I would like to see some other black holes added later on, most of which will be acting properly.

			choice
				`	"So this is not the first expedition here?"`

> 					goto astralfirst

Reviewer 2: This allows a loop. I'd remove this choice.

Author: Removed that choice, and replaced with another generic conversational item to keep the player involved, but doesn't go anywhere, so it is just a part of the flow of conversation.

				`	"How much longer does the equipment need?"`
			label astralfinish 

> 			`	"The equipment will be finished shortly, Captain <first>. I just need to make a a few observations and we will be able to depart. In the meantime, I would recommend against maintaining an active cloak or engaging any systems that consume fuel. For whatever reason, Sagittarius A* continues to be the one place that the stellar wind is utterly devoid of anything to collect as fuel."`

Reviewer 2: ```suggestion
			`	"The equipment will be finished shortly, Captain <first>. I just need to make a few observations and we will be able to depart. In the meantime, I would recommend against maintaining an active cloak or engaging any systems that consume fuel. For whatever reason, Sagittarius A* continues to be the one place that the stellar wind is utterly devoid of anything to collect as fuel."`
```

> 			`	"I will be going outside for a few minutes." Dawn declares. "I shall just be on the hull."`

Reviewer 2: ```suggestion
			`	"I will be going outside for a few minutes," Dawn declares. "I shall just be on the hull."`
```

			choice
				`	"Let me know when you are done."`
					goto astraldone
				`	"Would you mind if I came too?"`

> 			`	"Not at all, Captain. The view should be remarkable, and so long as we stay inside the shields, the radiation will not pose a problem." You quickly don your spacesuit, and join Dawn outside on the hull. As promised, the view is indeed magnificent. Dawn signs ocassionally, pointing out various features, and you appreciate that out here, sound would just be incongruous with the natural splendor.`

Reviewer 2: Sounds like a good place to put a scene.
```suggestion
			`	"Not at all, Captain. The view should be remarkable, and so long as we stay inside the shields, the radiation will not pose a problem." You quickly don your spacesuit and join Dawn outside on the hull. As promised, the view is indeed magnificent. Dawn signs occasionally, pointing out various features, and you appreciate that out here, sound would just be incongruous with the natural splendor.`
```

Author: Agreed. I'm making a note that I would like to put a scene here, but I don't want to hold up the PR as a whole while waiting for one.

			label astraldone

> 			`	After what feels like hours the equipment notifies you that it has finished taking its readings, and Dawn packs up the data in carefully shielded data crystals. Everything is ready to return to <destination>.`

Reviewer 2: ```suggestion
			`	After what feels like hours the, equipment notifies you that it has finished taking its readings, and Dawn packs up the data in carefully shielded data crystals. Everything is ready to return to <destination>.`
```

Author: This seems like an odd placement for a comma. Was it intended to be after the "hours" ? (I've changed this to add a comma after the hours.)

Reviewer 6: Hmm. I'm not seeing the comma in the actual line now. Did you add it?

Author: Thanks for pointing this out. I thought it got applied, but apparently didn't. Applied manually.

	on complete
		payment 500000
		conversation

> 			`The journey back to <planet> was blur as the <ship> jumped through countless systems. Gradually the brilliant density of stars that make up the inner core faded into the ominous and yet comfortingly familiar red glow of the Ember Waste. Finally you slide into the approach patterns of <planet>, quickly tapping out your code into the IFF system and watching as the protective gaze of the anti-air emplacements sweep the skies around you for shadowing craft.`

Reviewer 5: Good to see some acknowledgement of this brilliance.

Author: Thank you. I have been paying attention to all the descriptive talk that you and ravenshining and various others have been having about star density and the like. 🌟

> 			`The journey back to <planet> was blur as the <ship> jumped through countless systems. Gradually the brilliant density of stars that make up the inner core faded into the ominous and yet comfortingly familiar red glow of the Ember Waste. Finally you slide into the approach patterns of <planet>, quickly tapping out your code into the IFF system and watching as the protective gaze of the anti-air emplacements sweep the skies around you for shadowing craft.`

Author: OK. Changed it to `The journey back to <planet> was was a lesson in galactic formation as the <ship> jumped through a steady stream of systems.` That doesn't imply a mindset, and does hint that Dawn might be narrating it, or at least that <first> is paying attention to how the systems are changing; without actually implying a mindset.

	name "Preparation for expedition"
	description "Travel to <destination> to prepare for an expedition to Sagittarius A*. Ensure that you have at least six tons of outfit space available."
	passengers 1
	minor
	source
		government "Remnant"
	destination
		government "Remnant"
	to offer
		has "Remnant: Expanded Horizons Quarg 3: done"
		random < 25
	on offer
		conversation
			`You glance into the starport cafeteria and see Dawn sitting at a table looking worried. She hasn't noticed you yet. Do you approach her?`
			choice
				`	(Yes.)`
				`	(Not right now.)`
					defer
			`	She smiles when she sees you, and waves you to the seat opposite her. "Back from gallivanting around the galaxy with more tales to share?" she signs.`
			choice
				`	"Yes, as a matter of fact..."`
				`	"No, just hunting Korath..."`
				`	"No, just testing some stuff..."`
				`	"You look worried. Is something happening?"`
					goto worried
			`	She listens for a minute or two as you tell her about what you have been up to, but it is obvious that she has something on her mind.`
			choice
				`	"You look worried. Is everything alright?"`
				`	"You look distracted. Do you want to talk about it?"`
			label worried

> 			`	She makes a gesture of appreciation. "Yes, I'm worried, <first>. I keep feeling like there's something I'm missing." She frowns, contemplating something unseen. "I should back up a step. Since we last talked, I was assigned to serve with an astrophysics team for a year. My current work is part of the quantum condensate fabric density project, and I have been tasked with reviewing our bulk astrophysical data. It is fairly routine,  but I keep getting this prickly feeling that there's a problem that I am missing. Nothing obvious, of course. Our neural networks would pick up something obvious or repetitious really quickly. But the feeling just will not go away."`

Author: Thanks for pointing that out! I'll not commit this one to the batch just so I can keep it out to remind me to get to it in a few hours.

			choice
				`	"Have you asked someone with more experience for advice?"`
				`	"Don't worry about it."`
					goto worry
				`	"Can I help with something?"`
					goto help
			`	"I did. The senior researcher told me that most of the team feels that something is not quite right here. They say there are some potential theories, but nothing definitive, so they want me to investigate on my own without contaminating my thoughts with their ideas for now."`
				goto help
			label worry
			`	"Thanks for the advice, but that is our purpose here - to worry about all the stuff that humanity either cannot handle or that it turns a blind eye to. No, if there is something in the data to cause worry, we need to find it."`
			label help

> 			`	"If you would like to help, there is something you can do. I am currently putting together an expedition to Sagittarius A* to collect some fresh data I can use to check our records, and I could use a transport..."`

Author: Unpacking this a bit:
1) Yes, the Remnant do have JD, which is how they have sensor logs and recordings from previous visits.
2) She asks the player to assist because, well, the player is a known entity. She's already been on one research trip with the player, and that worked out nicely. 

What I was going for here is that she was in the processing of organizing an expedition, but hadn't arranged transportation yet. Since the player is here, and is a well known space traveler who has taken her on a previous research mission, then it makes sense to see if they are available to do so again.

The implication by the "and I could use a transport." isn't meant to imply that the *Remnant* need a transport, just that Dawn in particular does not have a JD equipped transport immediately available at her disposal.

			choice
				`	"I don't know where that is."`
					goto center
				`	"Isn't that the center of the galaxy?"`
					goto centeralt

> 				`	"I have been there."`

Author: I think I am going to opt for the player using complete words and not contractions. Call it an artifact of the sign language. It has its own ways of condensing symbology without chopping up and dropping parts of words.

				goto center
			label centeralt
			`	"Yes, that is correct." She signs firmly.`
			label center
			`	Her signs take on a rhythmic pace as if reciting something. "Sagittarius A* is a strong astronomical radio source and a supermassive black hole that is located at the center of the Milky Way. According to our best measurements, it has a mass of 3F6290 solar masses, and is surrounded by a ring of gas and dust that can range from 63 to 24D2 degrees Kelvin."`
			`	She returns to a conversational cadence. "Would you be interested in taking part in this expedition?"`
			choice
				`	"Sure, what do you need?"`
					goto scienceequipment
				`	"You want me to take you to a black hole?!"`
				`	"Sorry, I'm not interested in visiting a blackhole."`
					decline
			`	She looks momentarily surprised by your reaction. "Yes, exactly that. Current gravitational repulsor engines can nullify most of the gravitational pull, and an extra thermoelectric cooler or two should be sufficient to manage the heat. As such, the risks are mangeable so long as you do not try to approach the event horizon.`
			label scienceequipment

> 			`	"For this mission in specific, however, we will need some scientific equipment that is actually hardwired into your ship, not just carried along with us. When we arrive on <planet>, they will have the equipment ready to install. Just be aware that it will require six outfit space, and comes with a scientist, so you will need a free bunk for them, too.`

Author: I was actually thinking about her including you, the player, in that "we".

			`	"Once the laboratory module is installed, it will simply be a matter of travelling to Sagittarius A*. After our measurements, recordings, and observations are complete, we will return and report on our findings."`
			`	She glances down at her half-finished meal. "I am already packed and ready to go. I will meet you onboard the <ship> as soon as I am done eating."`
			branch dawnsymbiont
				has "remnant symbionts info"
			choice

> 				`	"Okay, I will meet you onboard the ship."`

Author: I personally prefer un-contracted statements, but I suppose we could go with it.

> 				`	"Okay, I will meet you onboard the ship."`

Reviewer 6: It's up to you. I think contractions just sound more natural, but it's subjective.

				accept
			label whatis
			apply
				"remnant symbionts info" += 1
			`	"Oh, no one has told you about them? I thought once you'd demonstrated your trustworthiness and earned your capital ship license you would have found out." She pauses to take another bite, then continues.`
				goto dawndetails
			label dawnsymbiont
			choice
				`	"Okay, I will meet you onboard the ship."`
					accept
				`	"Developed quite the appetite, haven't you?"`
			`	She nods. "Yeah, I am not yet used to the nutrient intake required to maintain my symbiont, so I have to consciously make sure that I eat enough to sustain it."`
			choice
				`	"Interesting. I'll see you onboard the <ship>."`
					accept
				`	"Could you tell me more about them?"`			
			label dawndetails
			apply
				"remnant symbionts info" += 1
			`	"Basically, ages ago we domesticated some of the creatures found here, and discovered that they could be attached to us to form a symbiotic relationship." She pauses again to eat. "I only recently received mine, so I am not fully adjusted to it yet."`
			choice
				`	"Okay, that's as much as I need to know."`
					goto onwards
				`	"What is it like to have a symbiont?"`

> 			`	"Oh, it is wonderful. It is very hard to describe. At first it is just slimy, then a lot of tingling. The tingling lasted for several hours, and then it was just like waking up: rising up out of darkness to a brighter world with just so much to see and learn. Suddenly, it was as if everything was moving slower and I could take a long hard look at everything..." Dawn is visibly excited, signing so fast you have trouble following.`

Reviewer 6: ```suggestion
			`	"Oh, it is wonderful. It is very hard to describe. At first it is just slimy, then a lot of tingling. The tingling lasted for several hours, and then it was just like waking up: rising up out of darkness to a brighter world with just so much to see and learn. Suddenly, it was as if everything was moving slower, and I could take a long, hard look at everything..." Dawn is visibly excited, signing so fast you have trouble following.`
```

			choice
				`	"Thanks for the description. I should go prep the <ship> now."`
					accept

> 				`	"Where is your symbiont? Yours does not seem to be obvious."`

Reviewer 6: ```suggestion
				`	"Where is your symbiont? Yours doesn't seem to be obvious."`
```

					goto dawnwhere
				`	"Could I see your symbiont?"`
			`	Dawn looks at you with an amused expression. "We don't know each other nearly well enough for that. Better to focus on the expedition. I will catch up to you on the ship."`
				accept
			label onwards
			`	She nods. "Well, I will finish my meal and meet you on the <ship> shortly." With that, she returns to her meal.`
				accept
			label dawnwhere
			`	Dawn looks at you with an amused expression. "Just as every human is a unique individual, so every symbiont is also unique. As such, they attach to different areas of the body. Mine is located primarily along my spine." She twists around, and you can see a small layer of the tell-tale scaly leather extending up the back of her neck.`
			`	"We should probably be focusing on the expedition, though. You go get the <ship> ready for launch, and I will be there shortly."`
				accept
	on accept
		event "remnant: lab availability"
	on complete
		outfit "Research Laboratory"
		conversation

> 			`You follow the directions of the starport control and settle onto a pad next to the outfitter. You barely have a chance to finish your landing checks before a team of Remnant have a compact pod waiting on your loading ramp. Once you open the cargo bay doors they pick it up and maneuver it inside. They start securing it to the ship while one approaches you and Dawn.`

Reviewer 6: ```suggestion
			`You follow the directions of the starport control and settle onto a pad next to the outfitter. You barely have a chance to finish your landing checks before a team of Remnant have a compact pod waiting on your loading ramp. Once you open the cargo bay doors, they pick it up and maneuver it inside. They start securing it to the ship while Dawn and one of the scientists approaches you.`
```
It isn't really established that Dawn is back before it seems like she just suddenly appears next to you. Having Dawn walk up with the scientist resolves the concern.

> 			`	"We have loaded the research laboratory onboard. The team will install it if they can find the space, but if they do not, it will be secured in the cargo bay." The Remnant turns to you as she continues. "Before you depart, please verify that the laboratory is installed in the correct location. You know your ship better that we do, after all." She turns back to Dawn and makes a gesture of respect. "Embers guide your path, Seeker." With that, she strides away into the hive of activity that is the outfitting bays.`

Reviewer 6: ```suggestion
			`	"We have loaded the research laboratory onboard. The team will install it if they can find the space, but if they do not, it will be secured in the cargo bay," The Remnant scientist explains. "Before you depart, please verify that the laboratory is installed in the correct location. You know your ship better that we do, after all." She turns back to Dawn and makes a gesture of respect. "Embers guide your path, Seeker." With that, she strides away into the hive of activity that is the outfitting bays.`
```
Edit for consistency with the suggestion above this one.

			choice

> 				`	"So, what is our next step?"`

Reviewer 6: ```suggestion
				`	"So, what's our next step?"`
```

					goto labnext
				`	"Seeker?"`

> 			`	Dawn looks a little embarassed. "Seeker is a title of respect for deep space explorers. It is unofficial - just a token of respect - but it is both a title that I aspire to earn and one I do not feel that I deserve."`

Reviewer 6: ```suggestion
			`	Dawn looks a little embarassed. "Seeker is a title of respect for deep space explorers. It is unofficial - just a token of respect - but it is a title that I both aspire to earn and do not feel I deserve yet."`
```
Personal preference for sentence flow.

			choice

> 				`	"I thought all Remnant were spacefarers?"`

Reviewer 6: ```suggestion
				`	"I thought all Remnant were spacefarers."`
```

Author: This is one of those odd statements that is also a question.

Reviewer 6: It's an [indirect question](https://www.thepunctuationguide.com/question-mark.html), so I think putting a question mark on the end is technically grammatically incorrect. Since we're dealing with dialogue, however, I think some leeway can be allowed on grammatical correctness for the sake of conveying tone. If you'd significantly prefer the question mark, this is probably a place where you can get away with it.

Author: If it is acceptable for me to get away with it here, I'd like to do that. :)

Reviewer 6: Go for it!

				`	"Okay. So what next?"`
					goto labnext

> 			`	"Oh, we are. Every Remnant has had some experience in space. But there is a big difference between a rotation of a month or two of duty in a spacefaring post and spending months or years drifting in the void surrounding distant systems."`

Reviewer 6: ```suggestion
			`	"Oh, we are. Every Remnant has had some experience in space. But there is a big difference between a few weeks of duty in a spacefaring post and spending months or years drifting in the void surrounding distant systems."`
```
More concise, and changing it to `weeks` helps differentiate it from `months or years` better. Up to you.

			choice
				`	"Years drifting in the void?"`
				`	"Okay, let's get on with this."`
					goto labnext

> 			`	"If you want good data, you need to spend time to get it. And especially when we are tracking stellar phenomenon, you need periods of long observation. Snapshots of the sky are good - that is what we will be collecting on this trip. But for watching the galaxy at large, other civilizations, and things of that ilk... Nothing is better than long-term observation." She looks lost in thought for a moment.`

Reviewer 6: ```suggestion
			`	"If you want good data, you need to spend time to get it. And especially when we are tracking stellar phenomenon, you need periods of long observation. Snapshots of the sky are good - that is what we will be collecting on this trip. But for watching the galaxy at large, other civilizations, and things of that ilk... Nothing is better than long-term observation." She looks lost in thought for a moment. "That is neither here nor there, however."`
```
I'd recommend moving `That is neither here nor there, however.` to this line. The "aside" is much longer here and more fitting of the phrase.

			label labnext

> 			`	"That is neither here nor there, however. Now that you have the research laboratory, you will need to locate a jump drive in order to reach Sagittarius A*." She pauses and checks her commlink. "If you do not already have one, you can salvage one from a Korath Raider. Our early warning system indicates that there are several incoming Korath fleets that should be here in the next day or two. Any one of them should have a jump drive you can take. Meet me in the starport once you have a jump drive."`

Reviewer 6: ```suggestion
			`	"Now that you have the research laboratory, you will need to locate a jump drive in order to reach Sagittarius A*." She pauses and checks her commlink. "If you do not already have one, you can salvage one from a Korath Raider. Our early warning system indicates that there are several incoming Korath fleets that should be here within the next day or two. Any one of them should have a jump drive you can take. Meet me in the starport once you have a jump drive."`
```

event "remnant: lab availability"
	outfitter "Remnant"
		add "Research Laboratory"

mission "Remnant: Expanded Horizons Astral 2"
	name "Research expedition to Sagittarius A*"
	description "Travel to Sagittarius A* with a research laboratory installed in order to make measurements on the supermassive black hole and then return to <destination>."
	blocked "You need at least 1 free bunk, a jump drive, and a research laboratory to take this mission."

> 	passengers 1

Reviewer 6: Okay, so if the outfit takes up a bunk, then isn't the person taking up that bunk Dawn (thus rendering the passenger requirement here unnecessary)? Or is there a separate scientist traveling with you?

Author: Hmmm... For the sake of the story, I think I'm going to tweak the requirements to say that it is Dawn taking up that spot.

In the future, though, that will be a generic "science officer" or whatever you prefer to call them.

Author: Made the adjustment to the passenger requirement (removing it from Astral 2), and added a comment in the conclusion from Dawn that she ran it for this mission, but if the player keeps it installed their ship will automatically hire a scientist to run it.

	waypoint "Sagittarius A*"
	source
		government "Remnant"
	destination
		government "Remnant"
	to offer
		has "Remnant: Expanded Horizons Astral 1: done"
	on offer
		require "Jump Drive"
		require "Research Laboratory"
		conversation

> 			`Dawn is just finishing her meal when you enter the spaceport cafeteria. She stands up as you approach, and deposits the tray in a receptacle next to her table. "Well, ready to go?"`

Reviewer 6: ```suggestion
			`Dawn is just finishing her meal when you enter the spaceport cafeteria. She stands up as you approach and deposits the tray in a receptacle next to her table. "Well, ready to go?"`
```

			choice
				`	"Yes."`
					goto astralonwards
				`	"I should double check everything."`
					defer
			`	She nods. "I am ready, so I will come with you and wait onboard the <ship> while you do your checks."`
			label astralonwards

> 			`	She gestures towards the exit closest to your ship, and heads off at a quick pace. Outside, she moves calmly through the bustle of the starport, stepping around quick-moving freight movers and camel wagons alike. Arriving at the <ship>, she turns, and waits for you to catch up.`

Reviewer 6: ```suggestion
			`	She gestures towards the exit closest to your ship and heads off at a quick pace. Outside, she moves calmly through the bustle of the starport, stepping around quick-moving freight movers and camel wagons alike. Arriving at the <ship>, she turns and waits for you to catch up.`
```

			`	Just as you arrive and reach for the console to open the airlock, you sense a change in your surroundings. Looking around, you see that a group of Remnant gathered around the two of you, focused on Dawn. A young Remnant - scarcely more than a child - steps forward. "Thank you for your patience," he says, then steps back. An elderly Remnant steps forward. "Thank you for your curiosity," she says, then steps back. A middle-aged Remnant steps forward. "Go with our blessing, Seeker, and know that wherever you go, the Remnant go with you." She says it simply, but you feel the weight of tradition behind them. She turns to you. "And you, stranger who walks the stars... May the Embers guide your steps as you walk among us." She nods respectfully, then steps back.`

> 			`	As soon as the third steps back there is a flurry of movement as each of the Remnant present quickly step forward, give her a quick hug, and then scurry off into the starport. Before you process what is going on all the Remnant gathered around you have vanished. You look at Dawn, and her expression is a mixture of awe and humility.`

Reviewer 6: ```suggestion
			`	As soon as the third steps back there is a flurry of movement as each of the Remnant present quickly step forward, give her a quick hug, and then scurry off into the starport. Before you process what is going on, all the Remnant gathered around you have vanished. You look at Dawn, and her expression is a mixture of awe and humility.`
```

			choice

> 				`	"They know you are coming back, right?"`

Reviewer 6: ```suggestion
				`	"They know you're coming back, right?"`
```

				`	"What was that about?"`
				`	"Alright, let's get going."`
					goto astralmoving
			`	"Well, what we are doing is dangerous. We will be crossing Korath territory, a wide expanse of uninhabited space, and then scanning a black hole. I have confidence that we will return, but nonetheless there is still a chance that we die in the process.`

> 			`	"Long ago we, as a people, decided that we must face the fear of separation, the unknown, and the danger. One of the ways we do that is by holding these informal ceremonies. When one of us is going to depart on their first long distance voyage into hostile territory, a group of Remnant gather to bid them farewell. It is a way to strengthen ourselves and remind each other of our connections."`

Reviewer 6: ```suggestion
			`	"Long ago we, as a people, decided that we must face the fear of separation, unknown, and danger. One of the ways we do that is by holding these informal ceremonies. When one of us is going to depart on their first long-distance voyage into hostile territory, a group of Remnant gather to bid them farewell. It is a way to strengthen ourselves and remind each other of our connections."`
```

			choice
				`	"Thanks for the explanation."`
					goto astralmoving
				`	"Kind of like a funeral, with friends and family gathering to see you off?"`

> 			`	"Oh, no. I said my farewells to my friends and relatives before we left. Those who gathered to see me off today were more of a representation of Remnant society. I only recognized a few of them, but they took the time to come down to the <ship> to see me before I left. They came to see me, and wish me well, so that no matter what challenges I face, I will know that the Remnant remember me and appreicate what I am doing." Her gestures fade into contemplative stillness for a moment, then resume motion.`

Reviewer 6: ```suggestion
			`	"Oh, no. I said my farewells to my friends and relatives before we left. Those who gathered to see me off today were more of a representation of Remnant society. I only recognized a few of them, but they took the time to come down to the <ship> to see me before I left. They came to see me and wish me well, so that no matter what challenges I face, I will know that the Remnant remember me and appreciate what I am doing." Her gestures fade into contemplative stillness for a moment, then resume motion.`
```

			label astralmoving
			`	She gestures towards the airlock door. "Lead the way, Captain."`
				accept
	on enter "Sagittarius A*"
		conversation

> 			`As the energy ripples from the jump drive fade, the spectacular void that is Sagittarius A* takes shape before you. Compared to the vast darkness of space that you are accustomed to, reality here feels almost like a color negative: even with the automatic filtering in your windows, space is incredibly bright with the densely packed stars that provide a backdrop for Sagittarius A*. The moment of awe is shattered by your alarms warning you of your impending doom as the <ship> begins to fall inwards towards the black hole. You quickly activate your repulsor engines, which, although vastly under-powered for dealing with the immense pull of a black hole, does provide a degree of protection so long as you maintain a safe distance from it.`

Reviewer 6: ```suggestion
			`As the energy ripples from the jump drive fade, the spectacular void that is Sagittarius A* takes shape before you. Compared to the vast darkness of space that you are accustomed to, reality here feels almost like a color negative: even with the automatic filtering in your windows, space is incredibly bright with the densely packed stars that provide a backdrop for Sagittarius A*. The moment of awe is shattered by alarms warning of your impending doom as the <ship> begins to fall inwards towards the black hole. You quickly activate your repulsor engines, which, although vastly under-powered for dealing with the immense pull of a black hole, does provide a degree of protection so long as you maintain a safe distance from it.`
```

> reality here feels almost like a color negative

Very nice line.

			`	After a few minutes of just absorbing the spectacle before you, Dawn pulls herself away from the cockpit windows and scurries down to the lab. Leaving you to your thoughts, she quickly has the lab set up and running a variety of recordings, test, and observation equipment. After a few hours of work, she reappears on the bridge.`

> 			`	"Things are proceeding smoothly, Captain. All the sensors are working nominally and we are creating extensive archives of astronomical data as well. Another few hours and we should have all the data we need."`

Reviewer 6: ```suggestion
			`	"Things are proceeding smoothly, Captain. All the sensors are working nominally, and we are creating extensive archives of astronomical data as well. Another few hours, and we should have all the data we need."`
```

			choice
				`	"Good work. What kind of data are you recording?`

> 				`	"Sounds good. Just let me know when you are ready to leave."`

Reviewer 6: ```suggestion
				`	"Sounds good. Just let me know when you're ready to leave."`
```

					goto astraldone

> 			`	She smiles, and replies. "It might be easier to ask what data we are leaving out. The data we are collecting broadly falls into three data sets: data on Sagittarius A* itself, data on its environment, and observations of the surrounding galaxy from here. Obviously, we cannot go take samples of Sagittarius A* itself, nor take direct measurements of most of its properties, but we can get data from increasingly minute distances from the event horizon. We can then add this data to that gathered by previous expeditions, note the differences, and use it in our modeling."`

Reviewer 6: ```suggestion
			`	She smiles, and replies. "It might be easier to ask what we are leaving out. The data we are collecting broadly falls into three sets: data on Sagittarius A* itself, data on its environment, and observations of the surrounding galaxy from here. Obviously, we cannot go take samples of Sagittarius A* itself, nor can we take direct measurements of most of its properties, but we can get data from increasingly minute distances from the event horizon. We can then add this data to that gathered by previous expeditions, note the differences, and use it in our modeling."`
```
Lots of `data` here. If you can think of anywhere else (either in this exact line or the entire conversation) that you could minimize usage, it definitely wouldn't hurt. [This might help, if you'd like.](https://www.thesaurus.com/browse/data)

			choice
				`	"Anything unusual in the data?"`
				`	"How much longer does the equipment need?"`
					goto astralfinish
			label astraldata
			`	"There are many oddities about this place. Every visit helps revise our physics models a little closer. But they also reveal problems." She frowns, her gestures taking on a curious tone. "For instance, you will note that your ramscoops are not picking up anything. According to standard astronomical theory, this system should be a veritable river of suitable material, and yet there is nothing. Not even the normal background trace amounts we find in deep space. We have visited other black holes, and they conform to the standard models in this respect. But not Sagittarius A*." She pauses, thinking about it for a moment.`
			choice
				`	"Is the equipment loaded?"`
				`	"How much longer does the equipment need?"`
			label astralfinish 

> 			`	"The equipment will be finished shortly, Captain <first>. I just need to make a few observations and we will be able to depart. In the meantime, I would recommend against maintaining an active cloak or engaging any systems that consume fuel. For whatever reason, Sagittarius A* continues to be the one place that the stellar wind is utterly devoid of anything to collect as fuel."`

Reviewer 6: ```suggestion
			`	"The equipment will be finished shortly, Captain <first>. I just need to make a few observations, and then we will be able to depart. In the meantime, I would recommend against maintaining an active cloak or engaging any systems that consume fuel. For whatever reason, Sagittarius A* continues to be the one place that the stellar wind is utterly devoid of anything to collect as fuel."`
```

			`	"I will be going outside for a few minutes," Dawn declares. "I shall just be on the hull."`
			choice
				`	"Let me know when you are done."`
					goto astraldone
				`	"Would you mind if I came too?"`

> 			`	"Not at all, Captain. The view should be remarkable, and so long as we stay inside the shields, the radiation will not pose a problem." You quickly don your spacesuit and join Dawn outside on the hull. As promised, the view is indeed magnificent. Dawn signs occasionally, pointing out various features, and you appreciate that out here, sound would just be incongruous with the natural splendor.`

Reviewer 6: They got some high-tech stuff if you can make out exact finger movements in spacesuits well enough for sign language to work. Just a random thought. Neat!

			label astraldone
			`	After what feels like hours the equipment notifies you that it has finished taking its readings, and Dawn packs up the data in carefully shielded data crystals. Everything is ready to return to <destination>.`
	on complete
		payment 500000
		conversation

> 			`The journey back to <planet> was was a lesson in galactic formation as the <ship> jumped through a steady stream of systems. Gradually the brilliant density of stars that make up the inner core faded into the ominous and yet comfortingly familiar red glow of the Ember Waste. Finally you slide into the approach patterns of <planet>, quickly tapping out your code into the IFF system and watching as the protective gaze of the anti-air emplacements sweep the skies around you for shadowing craft.`

Reviewer 6: ```suggestion
			`The journey back to <planet> was a lesson in galactic formation as the <ship> jumped through a steady stream of systems. Gradually, the brilliant density of stars that make up the inner core faded into the ominous and yet comfortingly familiar red glow of the Ember Waste. Finally, you slide into the approach patterns of <planet>, quickly tapping out your code into the IFF system and watching as the protective gaze of the anti-air emplacements sweep the skies around you for shadowing craft.`
```

> 			`You settle onto your assigned pad, and within seconds Remnant technicians have resupply cables hooked up, while a cheerful group greet Dawn and help her unload her luggage and data crystals. "Well, <first>, it has been a good trip." She taps at her commlink. "I have transferred our expedition's logistics allotment of <payment> to you. And in case you're wondering what to do with your research laboratory, you can keep it. We always need fresh data, and if you are interested, you can check up on places designated in the job board." She nods her head in respect to you. "Thank you, and may the Embers watch over you." She turns and heads off into the flows of people.`

Reviewer 6: ```suggestion
			`You settle onto your assigned pad, and within seconds Remnant technicians have resupply cables hooked up while a cheerful group greet Dawn before helping her unload her luggage and data crystals. "Well, <first>, it has been a good trip." She taps at her commlink. "I have transferred our expedition's logistics allotment of <payment> to you. And in case you're wondering what to do with your research laboratory, you can keep it. We always need fresh data, and if you are interested, you can check up on places designated in the job board." She nods her head in respect to you. "Thank you, and may the Embers watch over you." She turns and heads off into the flows of people.`
```
Just avoids the strange double `and` case that seems grammatically incorrect but is actually correct.

mission "Remnant: Expanded Horizons Astral job"
	name "Research Expedition to <waypoints>"
	description "Travel to <waypoints> with a research laboratory to collect data, then return to <destination>."
	job
	repeat
	waypoint
		not attributes "pleiades"

> 	to offer

Reviewer 6: Maybe a `random` element would be a good idea here? Up to you, just wanted to bring it up in case it hasn't been considered.

Author: Yes, I think I'll put in a random element here. It'll be a fairly high on - 50% or so. Collecting updated data is a very routine task.

				goto help
			label quantumcondensate
			`	"It is basically a project examining the density of Bose–Einstein condensates as correlated to the revised relativity paradigm's conceptualization of the fabric of space-time. It has a lot of potential implications, particularly for ramscoop, scanner, and weapons technology. At least, that is the hope. For now, though, I am trying to figure out what is off about this data."`
			choice
				`	"Have you asked someone with more experience for advice?"`
				`	"Don't worry about it."`
					goto worry
				`	"Can I help with something?"`
					goto help
			`	"I did. The senior researcher told me that most of the team feels something is not quite right here. They say there are some potential theories, but nothing definitive, so they want me to investigate on my own without contaminating my thoughts with their ideas for now."`
				goto help
			label worry
			`	"Thanks for the advice, but that is our purpose here - to worry about all the things that humanity either cannot handle or that it turns a blind eye to. No, if there is something in the data to cause worry, we need to find it."`
			label help

> 			`	"If you would like to help, there is something you can do. I am currently putting together an expedition to Sagittarius A* to collect some fresh data I can use to check our records, and I could use a transport..."`

Reviewer 7: ```suggestion
			`	"If you would like to help, there is something you can do. I am currently putting together an expedition to Sagittarius A* to collect some fresh data I can use to check our records, and I could use a transport."`
```

In general avoid ending dialog with ellipses.

Author: OK, I'll keep that in mind. I think the ellipses convey the intent, so I have added a bit after this, so it now ends:

> and I could use a transport..." She looks at you hopefully.

Does this work better?

				goto center
			label centeralt
			`	"Yes, that is correct." She signs firmly.`
			label center
			`	Her signs take on a rhythmic pace as if reciting something. "Sagittarius A* is a strong astronomical radio source and a supermassive black hole that is located at the center of the Milky Way. According to our best measurements, it has a mass of 3F6290 solar masses and is surrounded by a ring of gas and dust that can range from 63 to 24D2 degrees Kelvin."`
			`	She returns to a conversational cadence. "Would you be interested in taking part in this expedition?"`
			choice
				`	"Sure, what do you need?"`
					goto scienceequipment
				`	"You want me to take you to a black hole?!"`
				`	"Sorry, I'm not interested in visiting a black hole."`
					decline
			`	She looks momentarily surprised by your reaction. "Yes, exactly that. Current gravitational repulsor engines can nullify most of the gravitational pull, and an extra thermoelectric cooler or two should be sufficient to manage the heat. As such, the risks are mangeable so long as you do not try to approach the event horizon.`
			label scienceequipment

> 			`	"For this mission in specific, however, we will need some scientific equipment that is actually hardwired into your ship, not just carried along with us. When we arrive on <planet>, they will have the equipment ready to install. Just be aware that it will require six outfit space, and requires a scientist, so you will need a free bunk for them, too.`

Reviewer 7: ```suggestion
			`	"For this mission in specific, however, we will need some scientific equipment that is actually hardwired into your ship, not just carried along with us. When we arrive on <planet>, they will have the equipment ready to install. Just be aware that it will require six tons of outfit space and a scientist to run it, so you will need a free bunk for them, too.`
```

			`	"Once the laboratory module is installed, it will simply be a matter of traveling to Sagittarius A*. After our measurements, recordings, and observations are complete, we will return and report on our findings."`
			`	She glances down at her half-finished meal. "I am already packed and ready to go. I will meet you onboard the <ship> as soon as I am done eating."`
			branch dawnsymbiont
				has "remnant symbionts info"
			choice
				`	"Okay, I will meet you onboard the ship."`
					accept
				`	"Developed quite the appetite, haven't you?"`
			branch dawnlicense
				has "license: Remnant Capital"

> 			`	She shrugs. "I burn a lot of calories, and eating healthy is important. Besides, I don't know how good food will be on your ship." She resumes eating with vigor and does not seem inclined to continue the conversation.`

Reviewer 7: ```suggestion
			`	She shrugs. "I burn a lot of calories, and healthy eating is important. Besides, I don't know how good food will be on your ship." She resumes eating with vigor and does not seem inclined to continue the conversation.`
```

("eating healthy" is colloquial but grammatically incorrect, and her speech patterns indicate a preference for precision)

event "remnant: lab availability"
	outfitter "Remnant"
		add "Research Laboratory"

mission "Remnant: Expanded Horizons Astral 2"
	name "Research expedition to Sagittarius A*"
	description "Travel to Sagittarius A* with a research laboratory installed in order to make measurements on the supermassive black hole and then return to <destination>."
	blocked "You need a jump drive and a research laboratory to take this mission."
	waypoint "Sagittarius A*"
	source
		government "Remnant"
	destination
		government "Remnant"
	to offer
		has "Remnant: Expanded Horizons Astral 1: done"
	on offer
		require "Jump Drive"
		require "Research Laboratory"
		conversation
			`Dawn is just finishing her meal when you enter the spaceport cafeteria. She stands up as you approach and deposits the tray in a receptacle next to her table. "Well, ready to go?"`
			choice
				`	"Yes."`
					goto astralonwards
				`	"I should double check everything."`
					defer
			`	She nods. "I am ready, so I will come with you and wait onboard the <ship> while you do your checks."`
			label astralonwards

> 			`	She gestures towards the exit closest to your ship and heads off at a quick pace. Outside, she moves calmly through the bustle of the starport, stepping around quick-moving freight movers and camel wagons alike. Arriving at the <ship>, she turns and waits for you to catch up.`

Reviewer 7: ```suggestion
			`	She gestures towards the exit closest to your ship and heads off at a quick pace. Outside, she moves calmly through the hustle and bustle of the starport, stepping around quick-moving freight movers and camel wagons alike. Arriving at the <ship>, she turns and waits for you to catch up.`
```

			choice
				`	"Thanks for the explanation."`
					goto astralmoving
				`	"Kind of like a funeral, with friends and family gathering to see you off?"`
			`	"Oh, no. I said my farewells to my friends and relatives before we left. Those who gathered to see me off today were more of a representation of Remnant society. I only recognized a few of them, but they took the time to come down to the <ship> to meet me before I left. They came to see me and wish me well, so that no matter what challenges I face, I will know that the Remnant remember me and appreciate what I am doing." Her gestures fade into contemplative stillness for a moment, then resume motion.`
			label astralmoving
			`	She gestures towards the airlock door. "Lead the way, Captain."`
				accept
	on enter "Sagittarius A*"
		conversation

> 			`As the energy ripples from the jump drive fade, the spectacular void that is Sagittarius A* takes shape before you. Compared to the vast darkness of space that you are accustomed to, reality here feels almost like a color negative: even with the automatic filtering in your windows, space is incredibly bright with the densely packed stars that provide a backdrop for Sagittarius A*. The moment of awe is shattered by alarms warning of your impending doom as the <ship> begins to fall inwards towards the black hole. You quickly activate your repulsor engines, which, although vastly under-powered for dealing with the immense pull of a black hole, does provide a degree of protection so long as you maintain a safe distance from it.`

Reviewer 7: ```suggestion
			`As the jump drive's energy ripples fade away, the spectacular void that is Sagittarius A* takes shape before you. Compared to the vast darkness of space that you are accustomed to, reality here feels almost like a color negative: even with the automatic filtering in your windows, space is incredibly bright with the densely packed stars that provide a backdrop for its inky mass. The moment of awe is shattered by alarms warning of your impending doom as the <ship> begins to fall inwards towards the black hole. You quickly activate your repulsor engines, which, although vastly under-powered for dealing with the immense pull of a black hole, does provide a degree of protection so long as you maintain a safe distance from it.`
```

> 			`	After a few minutes of just absorbing the spectacle before you, Dawn pulls herself away from the cockpit windows and scurries down to the lab. Leaving you to your thoughts, she quickly has the lab set up and running a variety of recordings, test, and observation equipment. After a few hours of work, she reappears on the bridge.`

Reviewer 7: ```suggestion
			`	After a few minutes of just absorbing the spectacle before you, Dawn pulls herself away from the cockpit windows and scurries down to the lab. Leaving you to your thoughts, she quickly has the lab set up and running a variety of recordings, tests, and observations. After a few hours of work, she reappears on the bridge.`
```

			`	"Things are proceeding smoothly, Captain. All the sensors are working nominally, and we are creating extensive archives of astronomical data as well. Another few hours, and we should have all the data we need."`
			choice
				`	"Good work. What kind of data are you recording?`

> 				`	"Sounds good. Just let me know when you are ready to leave."`

Reviewer 7: ```suggestion
				`	"Sounds good. Just let me know when you're ready to leave."`
```

Author: I am avoiding using contractions in all Remnant dialog, for for all of the player's dialog that takes place in sign. This is a stylistic issue which I am attributing to 
a) a throwback to their ancestors which were overwhelmingly either academics or starship crew, and in either case both tend to depend on precision, and be made up of educated people who tend not to concatenate.
b) a linguistic oddity of sign language that, particularly in view of those who developed it, doesn't concatenate words.

As a result of these two factors (especially the second), the pilot would be forced into the same pattern.

			choice
				`	"Is the equipment finished?"`
				`	"How much longer does the equipment need?"`
			label astralfinish 
			`	"The equipment will be finished shortly, Captain <first>. I just need to make a few observations, and then we will be able to depart. In the meantime, I would recommend against maintaining an active cloak or engaging any systems that consume fuel. For whatever reason, Sagittarius A* continues to be the one place that the stellar wind is utterly devoid of anything to collect as fuel."`
			`	"I will be going outside for a few minutes," Dawn declares. "I shall just be on the hull."`
			choice

> 				`	"Let me know when you are done."`

Reviewer 7: ```suggestion
				`	"Let me know when you're done."`
```

Author: Same as the previous contraction. Long story short, I am deliberately avoiding using contractions for both sides of Remnant dialog.

I realize this does make it sound slightly odd, because we are used to having those contractions there. This oddity is intentional, and is a subtle hint that their thought patterns are at least a _little_ different.

					goto astraldone
				`	"Would you mind if I came too?"`
			`	"Not at all, Captain. The view should be remarkable, and so long as we stay inside the shields, the radiation will not pose a problem." You quickly don your spacesuit and join Dawn outside on the hull. As promised, the view is indeed magnificent. Dawn signs occasionally, pointing out various features, and you appreciate that out here, sound would just be incongruous with the natural splendor.`
			label astraldone
			`	After what feels like hours, the equipment notifies you that it has finished taking its readings, and Dawn packs up the data in carefully shielded data crystals. Everything is ready to return to <destination>.`
	on complete
		payment 500000
		conversation

> 			`The journey back to <planet> was a lesson in galactic formation as the <ship> jumped through a steady stream of systems. Gradually, the brilliant density of stars that make up the inner core faded into the ominous and yet comfortingly familiar red glow of the Ember Waste. Finally, you slide into the approach patterns of <planet>, quickly tapping out your code into the IFF system and watching as the protective gaze of the anti-air emplacements sweep the skies around you for shadowing craft.`

Reviewer 7: ```suggestion
			`The journey back to <planet> was a lesson in galactic formation as the <ship> jumped through a steady stream of systems. Gradually, the brilliant density of stars that make up the inner core faded into the ominous and yet comfortingly familiar red glow of the Ember Waste. Finally, you slide into the approach patterns of <planet>, quickly tapping out your code into the IFF system and watching as the anti-air emplacements sweep the skies around you for shadowing craft with their protective gaze.`
```

Author: I'm not sure about this change. While the original feels very evocative, the suggestion makes it feel like the `with their protective gaze.` is just an extraneous run-on that should be chopped off.

Reviewer 7: I suggested this change because the original is not grammatically correct; you're having "the protected gaze" being the thing that sweeps the sky. Protective gazes don't sweep the sky, but anti-aircraft emplacements do.

Author: As it happens, [Collin's Dictionary](https://www.collinsdictionary.com/dictionary/english/sweep) actually uses something very similar to this example. See number 11

> Her gaze sweeps rapidly around the room.

Reviewer 7: You learn something every day!

> 			`You settle onto your assigned pad, and within seconds Remnant technicians have resupply cables hooked up while a cheerful group greet Dawn before helping her unload her luggage and data crystals. "Well, <first>, it has been a good trip." She taps at her commlink. "I have transferred our expedition's logistics allotment of <payment> to you. And in case you're wondering what to do with your research laboratory, you can keep it. We always need fresh data, and if you are interested, you can check up on places designated in the job board. While I handled the science on this expedition, if you keep it installed your ship systems will automatically hire a scientist to run it." She nods her head in respect to you. "Thank you, and may the Embers watch over you." She turns and heads off into the flows of people.`

Reviewer 7: ```suggestion
			`You settle onto your assigned pad, and within seconds Remnant technicians have resupply cables hooked up while a cheerful group greets Dawn before helping her unload her luggage and data crystals. "Well, <first>, it has been a good trip." She taps at her commlink. "I have transferred our expedition's logistics allotment of <payment> to you. And in case you're wondering what to do with your research laboratory, you can keep it. We always need fresh data, and if you are interested, you can check up on places designated in the job board. While I handled the science on this expedition, if you keep it installed your ship systems will automatically hire a scientist to run it." She nods her head in respect to you. "Thank you, and may the Embers watch over you." She turns and heads off into the flows of people.`
```

> mission "Remnant: Expanded Horizons Astral job"

Reviewer 7: This mission could benefit from a little dialog that shows up when you enter the target system so you know you've done what's needed.

Author: Thanks for suggesting it. I have now added a little something here.

			`	"Once the laboratory module is installed, it will simply be a matter of traveling to Sagittarius A*. After our measurements, recordings, and observations are complete, we will return and report on our findings."`
			`	She glances down at her half-finished meal. "I am already packed and ready to go. I will meet you onboard the <ship> as soon as I am done eating."`
			branch dawnsymbiont
				has "remnant symbionts info"
			choice
				`	"Okay, I will meet you onboard the ship."`
					accept
				`	"Developed quite the appetite, haven't you?"`
			branch dawnlicense
				has "license: Remnant Capital"
			`	She shrugs. "I burn a lot of calories, and eating healthy is important. Besides, I don't know how good food will be on your ship." She resumes eating with vigor and does not seem inclined to continue the conversation.`
				accept
			label dawnlicense
			`	"I am not yet used to the nutrient intake required to maintain my symbiont, so I have to consciously make sure that I eat enough to sustain it."`
			choice
				`	"Symbiont? What is that?"`
					goto whatis
				`	"What? You're joking, right?"`

> 			`	"Suit yourself. I will meet you onboard once I'm done." She shrugs, and returns to eating."`

Reviewer 8: I don't see how this follows from making the above choice... seems like two different conversations have been joined.

Author: This is referring to the "Suit yourself." response?

Basically, Dawn's not interested in convincing <first> of anything, nor does she feel the need to convince them of anything. If <first> is going to treat it like a joke or something unbelievable, then Dawn is just going to drop the subject.

Reviewer 8: Yeah, I gathered that was the intent but it just sounds odd.

Reviewer 8: I'm fine with this being left as is.

			choice
				`	"So, what is our next step?"`
					goto labnext
				`	"Seeker?"`
			`	Dawn looks a little embarassed. "Seeker is a title of respect for deep space explorers. It is unofficial - just a token of respect - but it is a title that I both aspire to earn and do not feel I deserve yet."`
			choice
				`	"I thought all Remnant were spacefarers?"`
				`	"Okay. So what next?"`
					goto labnext
			`	"Oh, we are. Every Remnant has had some experience in space. But there is a big difference between a few weeks of duty in a spacefaring post and spending months or years drifting in the void surrounding distant systems."`
			choice
				`	"Years drifting in the void?"`
				`	"Okay, let's get on with this."`
					goto labnext

> 			`	"If you want good data, you need to spend time to get it. And especially when we are tracking stellar phenomenon, you need periods of long observation. Snapshots of the sky are good - that is what we will be collecting on this trip. But for watching the galaxy at large, other civilizations, and things of that ilk... Nothing is better than long-term observation." She looks lost in thought for a moment. "That is neither here nor there, however."`

Reviewer 2: ```suggestion
			`	"If you want good data, you need to spend time to get it. And especially when we are tracking stellar phenomenon, you need periods of long observation. Snapshots of the sky are good - that is what we will be collecting on this trip. But for watching the galaxy at large, other civilizations, and things of that ilk... Nothing is better than long-term observation." She looks lost in thought for a moment. "That is neither here nor there, however.`
```

			label labnext
			`	"Now that you have the research laboratory, you will need to locate a jump drive in order to reach Sagittarius A*." She pauses and checks her commlink. "If you do not already have one, you can salvage one from a Korath Raider. Our early warning system indicates that there are several incoming Korath fleets that should be here within the next day or two. Any one of them should have a jump drive you can take. Meet me in the starport once you have a jump drive."`

event "remnant: lab availability"
	outfitter "Remnant"
		add "Research Laboratory"

mission "Remnant: Expanded Horizons Astral 2"
	name "Research expedition to Sagittarius A*"
	description "Travel to Sagittarius A* with a research laboratory installed in order to make measurements on the supermassive black hole and then return to <destination>."

> 	blocked "You need at least 1 free bunk, a jump drive, and a research laboratory to take this mission."

Reviewer 8: I don't see the requirement for a free bunk nor the passengers keyword.

Reviewer 8: Never mind.  I see that the lab requires 1 crew.

Author: Hmmm... Right now, Dawn is taking the place of the crew member that would go with the research laboratory, so this is actually an oversight. (There used to be both an extra crew member & Dawn). So I'll remove that bunk mention.

			`	As soon as the third steps back there is a flurry of movement as each of the Remnant present quickly step forward, give her a quick hug, and then scurry off into the starport. Before you process what is going on, all the Remnant gathered around you have vanished. You look at Dawn, and her expression is a mixture of awe and humility.`
			choice
				`	"They know you are coming back, right?"`
				`	"What was that about?"`
				`	"Alright, let's get going."`
					goto astralmoving

> 			`	"Well, what we are doing is dangerous. We will be crossing Korath territory, a wide expanse of uninhabited space, and then scanning a black hole. I have confidence that we will return, but nonetheless there is still a chance that we die in the process.`

Reviewer 8: `that we die` doesn't sound right to me.  I think it should be `that we could die`

Author: I agree, `there is still a chance that we could die in the process" does sound more natural than "there is still a chance that we die in the process."

That being said, I think it is still technically correct; and I like the slightly more formal/unnatural framing for it. The Player and Dawn are going through a process where death is more than a mere possibility. The way I have it framed makes death sound a bit more likely. It isn't a chance that they will be in a situation where death might occur. They *will* be in a situation where death is a possibility. The unknown factor is whether they die, not whether they are in a situation where they *could* die.

Reviewer 8: What about *will* die, then?

Author: That could work. I'll change that over and see how people like it.

			`	"Long ago we, as a people, decided that we must face the fear of separation, unknown, and danger. One of the ways we do that is by holding these informal ceremonies. When one of us is going to depart on their first long-distance voyage into hostile territory, a group of Remnant gather to bid them farewell. It is a way to strengthen ourselves and remind each other of our connections."`
			choice
				`	"Thanks for the explanation."`
					goto astralmoving
				`	"Kind of like a funeral, with friends and family gathering to see you off?"`

> 			`	"Oh, no. I said my farewells to my friends and relatives before we left. Those who gathered to see me off today were more of a representation of Remnant society. I only recognized a few of them, but they took the time to come down to the <ship> to see me before I left. They came to see me and wish me well, so that no matter what challenges I face, I will know that the Remnant remember me and appreciate what I am doing." Her gestures fade into contemplative stillness for a moment, then resume motion.`

Reviewer 8: She says they came to see her twice across 2 sentences... a bit redundant, isn't it?

Author: It is a bit, but I was going for a sort of shift in tone here. The first sentence is basically a casual sort of thing, whereas the second is more of a formal statement of intent, kind of like how people go to see someone at a wake or funeral; or in the sense of the North African tribes statement/greeting of "I see you." (https://www.lollydaskal.com/leadership/i-see-you/)

Reviewer 8: Again, I understood that after multiple readings.  I wonder if something could be added in between to ease the shift and separate the duplicate statements a little.

Author: Changing the first "see" to "meet" fits this, I think.

			`	"Things are proceeding smoothly, Captain. All the sensors are working nominally, and we are creating extensive archives of astronomical data as well. Another few hours, and we should have all the data we need."`
			choice
				`	"Good work. What kind of data are you recording?`
				`	"Sounds good. Just let me know when you are ready to leave."`
					goto astraldone
			`	She smiles, and replies. "It might be easier to ask what we are leaving out. The data we are collecting broadly falls into three sets: data on Sagittarius A* itself, data on its environment, and observations of the surrounding galaxy from here. Obviously, we cannot go take samples of Sagittarius A* itself, nor can we take direct measurements of most of its properties, but we can get data from increasingly minute distances from the event horizon. We can then add this data to that gathered by previous expeditions, note the differences, and use it in our modeling."`
			choice
				`	"So this is not the first expedition here?"`
				`	"Anything unusual in the data?"`
					goto astraldata
				`	"How much longer does the equipment need?"`
					goto astralfinish

> 			label astralfirst

Reviewer 8: Is this label going to be used somewhere?

Author: A relic of an early version that had a loop. Removed.

			choice
				`	"Anything unusual in the data?"`
				`	"How much longer does the equipment need?"`
					goto astralfinish
			label astraldata
			`	"There are many oddities about this place. Every visit helps revise our physics models a little closer. But they also reveal problems." She frowns, her gestures taking on a curious tone. "For instance, you will note that your ramscoops are not picking up anything. According to standard astronomical theory, this system should be a veritable river of suitable material, and yet there is nothing. Not even the normal background trace amounts we find in deep space. We have visited other black holes, and they conform to the standard models in this respect. But not Sagittarius A*." She pauses, thinking about it for a moment.`
			choice

> 				`	"Is the equipment loaded?"`

Reviewer 8: Shouldn't this be `equipment finished`?

Author: Yes. Fixed. Thanks!
